{"cells":[{"metadata":{},"cell_type":"markdown","source":"### Install Libraries and Models "},{"metadata":{"trusted":true,"scrolled":true},"cell_type":"code","source":"#Install needed packages and NLP models\n!pip install -U pysolr\n!pip install -U scispacy\n!pip install -U jsonpath-ng\n!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_core_sci_md-0.2.4.tar.gz\n!pip install -U pyvis\n    \n!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_craft_md-0.2.4.tar.gz\n!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_jnlpba_md-0.2.4.tar.gz\n!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_bc5cdr_md-0.2.4.tar.gz\n!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_bionlp13cg_md-0.2.4.tar.gz","execution_count":2,"outputs":[{"output_type":"stream","text":"Collecting pysolr\n  Using cached pysolr-3.8.1-py2.py3-none-any.whl (16 kB)\nRequirement already satisfied, skipping upgrade: requests>=2.9.1 in /opt/conda/lib/python3.6/site-packages (from pysolr) (2.22.0)\nRequirement already satisfied, skipping upgrade: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /opt/conda/lib/python3.6/site-packages (from requests>=2.9.1->pysolr) (1.24.3)\nRequirement already satisfied, skipping upgrade: chardet<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from requests>=2.9.1->pysolr) (3.0.4)\nRequirement already satisfied, skipping upgrade: idna<2.9,>=2.5 in /opt/conda/lib/python3.6/site-packages (from requests>=2.9.1->pysolr) (2.8)\nRequirement already satisfied, skipping upgrade: certifi>=2017.4.17 in /opt/conda/lib/python3.6/site-packages (from requests>=2.9.1->pysolr) (2019.11.28)\nInstalling collected packages: pysolr\nSuccessfully installed pysolr-3.8.1\nProcessing /root/.cache/pip/wheels/80/01/69/37a2ab4f9b61773c187d83257ffb31365a5ad57e7779ae5e92/scispacy-0.2.4-py3-none-any.whl\nRequirement already satisfied, skipping upgrade: conllu in /opt/conda/lib/python3.6/site-packages (from scispacy) (1.3.1)\nCollecting awscli\n  Using cached awscli-1.18.38-py2.py3-none-any.whl (3.0 MB)\nCollecting pysbd\n  Using cached pysbd-0.2.3-py3-none-any.whl (24 kB)\nRequirement already satisfied, skipping upgrade: spacy>=2.2.1 in /opt/conda/lib/python3.6/site-packages (from scispacy) (2.2.3)\nCollecting nmslib>=1.7.3.6\n  Using cached nmslib-2.0.5-cp36-cp36m-manylinux2010_x86_64.whl (12.9 MB)\nRequirement already satisfied, skipping upgrade: joblib in /opt/conda/lib/python3.6/site-packages (from scispacy) (0.14.1)\nRequirement already satisfied, skipping upgrade: numpy in /opt/conda/lib/python3.6/site-packages (from scispacy) (1.18.1)\nRequirement already satisfied, skipping upgrade: scikit-learn>=0.20.3 in /opt/conda/lib/python3.6/site-packages (from scispacy) (0.22.2.post1)\nRequirement already satisfied, skipping upgrade: docutils<0.16,>=0.10 in /opt/conda/lib/python3.6/site-packages (from awscli->scispacy) (0.15.2)\nRequirement already satisfied, skipping upgrade: PyYAML<5.4,>=3.10; python_version != \"3.4\" in /opt/conda/lib/python3.6/site-packages (from awscli->scispacy) (5.3)\nCollecting rsa<=3.5.0,>=3.1.2\n  Using cached rsa-3.4.2-py2.py3-none-any.whl (46 kB)\nRequirement already satisfied, skipping upgrade: s3transfer<0.4.0,>=0.3.0 in /opt/conda/lib/python3.6/site-packages (from awscli->scispacy) (0.3.3)\nRequirement already satisfied, skipping upgrade: colorama<0.4.4,>=0.2.5; python_version != \"3.4\" in /opt/conda/lib/python3.6/site-packages (from awscli->scispacy) (0.4.3)\nCollecting botocore==1.15.38\n  Using cached botocore-1.15.38-py2.py3-none-any.whl (6.1 MB)\nRequirement already satisfied, skipping upgrade: murmurhash<1.1.0,>=0.28.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->scispacy) (1.0.2)\nRequirement already satisfied, skipping upgrade: thinc<7.4.0,>=7.3.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->scispacy) (7.3.1)\nRequirement already satisfied, skipping upgrade: setuptools in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->scispacy) (45.2.0.post20200210)\nRequirement already satisfied, skipping upgrade: plac<1.2.0,>=0.9.6 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->scispacy) (0.9.6)\nRequirement already satisfied, skipping upgrade: requests<3.0.0,>=2.13.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->scispacy) (2.22.0)\nRequirement already satisfied, skipping upgrade: wasabi<1.1.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->scispacy) (0.6.0)\nRequirement already satisfied, skipping upgrade: preshed<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->scispacy) (3.0.2)\nRequirement already satisfied, skipping upgrade: catalogue<1.1.0,>=0.0.7 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->scispacy) (1.0.0)\nRequirement already satisfied, skipping upgrade: blis<0.5.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->scispacy) (0.4.1)\nRequirement already satisfied, skipping upgrade: srsly<1.1.0,>=0.1.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->scispacy) (1.0.2)\nRequirement already satisfied, skipping upgrade: cymem<2.1.0,>=2.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->scispacy) (2.0.3)\nRequirement already satisfied, skipping upgrade: pybind11>=2.2.3 in /opt/conda/lib/python3.6/site-packages (from nmslib>=1.7.3.6->scispacy) (2.4.3)\nRequirement already satisfied, skipping upgrade: psutil in /opt/conda/lib/python3.6/site-packages (from nmslib>=1.7.3.6->scispacy) (5.6.7)\nRequirement already satisfied, skipping upgrade: scipy>=0.17.0 in /opt/conda/lib/python3.6/site-packages (from scikit-learn>=0.20.3->scispacy) (1.4.1)\nRequirement already satisfied, skipping upgrade: pyasn1>=0.1.3 in /opt/conda/lib/python3.6/site-packages (from rsa<=3.5.0,>=3.1.2->awscli->scispacy) (0.4.8)\nRequirement already satisfied, skipping upgrade: urllib3<1.26,>=1.20; python_version != \"3.4\" in /opt/conda/lib/python3.6/site-packages (from botocore==1.15.38->awscli->scispacy) (1.24.3)\nRequirement already satisfied, skipping upgrade: jmespath<1.0.0,>=0.7.1 in /opt/conda/lib/python3.6/site-packages (from botocore==1.15.38->awscli->scispacy) (0.9.5)\nRequirement already satisfied, skipping upgrade: python-dateutil<3.0.0,>=2.1 in /opt/conda/lib/python3.6/site-packages (from botocore==1.15.38->awscli->scispacy) (2.8.0)\nRequirement already satisfied, skipping upgrade: tqdm<5.0.0,>=4.10.0 in /opt/conda/lib/python3.6/site-packages (from thinc<7.4.0,>=7.3.0->spacy>=2.2.1->scispacy) (4.42.0)\nRequirement already satisfied, skipping upgrade: chardet<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->scispacy) (3.0.4)\nRequirement already satisfied, skipping upgrade: idna<2.9,>=2.5 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->scispacy) (2.8)\nRequirement already satisfied, skipping upgrade: certifi>=2017.4.17 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->scispacy) (2019.11.28)\nRequirement already satisfied, skipping upgrade: importlib-metadata>=0.20; python_version < \"3.8\" in /opt/conda/lib/python3.6/site-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->scispacy) (1.5.0)\nRequirement already satisfied, skipping upgrade: six>=1.5 in /opt/conda/lib/python3.6/site-packages (from python-dateutil<3.0.0,>=2.1->botocore==1.15.38->awscli->scispacy) (1.14.0)\nRequirement already satisfied, skipping upgrade: zipp>=0.5 in /opt/conda/lib/python3.6/site-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->scispacy) (2.2.0)\n\u001b[31mERROR: allennlp 0.9.0 has requirement spacy<2.2,>=2.1.0, but you'll have spacy 2.2.3 which is incompatible.\u001b[0m\nInstalling collected packages: rsa, botocore, awscli, pysbd, nmslib, scispacy\n  Attempting uninstall: rsa\n    Found existing installation: rsa 4.0\n    Uninstalling rsa-4.0:\n      Successfully uninstalled rsa-4.0\n  Attempting uninstall: botocore\n    Found existing installation: botocore 1.15.13\n    Uninstalling botocore-1.15.13:\n      Successfully uninstalled botocore-1.15.13\nSuccessfully installed awscli-1.18.38 botocore-1.15.38 nmslib-2.0.5 pysbd-0.2.3 rsa-3.4.2 scispacy-0.2.4\nProcessing /root/.cache/pip/wheels/c6/ff/e9/27e5651ea34a30a4e5caaebae638e120130bafb871d4d7a83f/jsonpath_ng-1.5.1-py3-none-any.whl\nRequirement already satisfied, skipping upgrade: ply in /opt/conda/lib/python3.6/site-packages (from jsonpath-ng) (3.11)\nRequirement already satisfied, skipping upgrade: decorator in /opt/conda/lib/python3.6/site-packages (from jsonpath-ng) (4.4.1)\nRequirement already satisfied, skipping upgrade: six in /opt/conda/lib/python3.6/site-packages (from jsonpath-ng) (1.14.0)\nInstalling collected packages: jsonpath-ng\nSuccessfully installed jsonpath-ng-1.5.1\nCollecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_core_sci_md-0.2.4.tar.gz\n  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_core_sci_md-0.2.4.tar.gz (70.0 MB)\n","name":"stdout"},{"output_type":"stream","text":"\u001b[K     |████████████████████████████████| 70.0 MB 860 kB/s eta 0:00:01\n\u001b[?25hRequirement already satisfied: spacy>=2.2.1 in /opt/conda/lib/python3.6/site-packages (from en-core-sci-md==0.2.4) (2.2.3)\nRequirement already satisfied: thinc<7.4.0,>=7.3.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (7.3.1)\nRequirement already satisfied: wasabi<1.1.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (0.6.0)\nRequirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (1.0.2)\nRequirement already satisfied: srsly<1.1.0,>=0.1.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (1.0.2)\nRequirement already satisfied: requests<3.0.0,>=2.13.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (2.22.0)\nRequirement already satisfied: numpy>=1.15.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (1.18.1)\nRequirement already satisfied: plac<1.2.0,>=0.9.6 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (0.9.6)\nRequirement already satisfied: catalogue<1.1.0,>=0.0.7 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (1.0.0)\nRequirement already satisfied: setuptools in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (45.2.0.post20200210)\nRequirement already satisfied: preshed<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (3.0.2)\nRequirement already satisfied: cymem<2.1.0,>=2.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (2.0.3)\nRequirement already satisfied: blis<0.5.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-core-sci-md==0.2.4) (0.4.1)\nRequirement already satisfied: tqdm<5.0.0,>=4.10.0 in /opt/conda/lib/python3.6/site-packages (from thinc<7.4.0,>=7.3.0->spacy>=2.2.1->en-core-sci-md==0.2.4) (4.42.0)\nRequirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-core-sci-md==0.2.4) (1.24.3)\nRequirement already satisfied: idna<2.9,>=2.5 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-core-sci-md==0.2.4) (2.8)\nRequirement already satisfied: chardet<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-core-sci-md==0.2.4) (3.0.4)\nRequirement already satisfied: certifi>=2017.4.17 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-core-sci-md==0.2.4) (2019.11.28)\nRequirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /opt/conda/lib/python3.6/site-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-core-sci-md==0.2.4) (1.5.0)\nRequirement already satisfied: zipp>=0.5 in /opt/conda/lib/python3.6/site-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-core-sci-md==0.2.4) (2.2.0)\nBuilding wheels for collected packages: en-core-sci-md\n  Building wheel for en-core-sci-md (setup.py) ... \u001b[?25ldone\n\u001b[?25h  Created wheel for en-core-sci-md: filename=en_core_sci_md-0.2.4-py3-none-any.whl size=70498245 sha256=6954f247cd535aedc209d885f3f734d34cf0a0bf5a016ea56e8e21900e44e3a6\n  Stored in directory: /root/.cache/pip/wheels/0b/d7/91/ad4d42842bdd001ccd505e492eba644c4c97b1fc5558c1d5ee\nSuccessfully built en-core-sci-md\nInstalling collected packages: en-core-sci-md\nSuccessfully installed en-core-sci-md-0.2.4\nRequirement already up-to-date: pyvis in /opt/conda/lib/python3.6/site-packages (0.1.7.0)\nRequirement already satisfied, skipping upgrade: networkx>=1.11 in /opt/conda/lib/python3.6/site-packages (from pyvis) (2.4)\nRequirement already satisfied, skipping upgrade: ipython>=5.3.0 in /opt/conda/lib/python3.6/site-packages (from pyvis) (7.12.0)\nRequirement already satisfied, skipping upgrade: jinja2>=2.9.6 in /opt/conda/lib/python3.6/site-packages (from pyvis) (2.11.1)\nRequirement already satisfied, skipping upgrade: decorator>=4.3.0 in /opt/conda/lib/python3.6/site-packages (from networkx>=1.11->pyvis) (4.4.1)\nRequirement already satisfied, skipping upgrade: traitlets>=4.2 in /opt/conda/lib/python3.6/site-packages (from ipython>=5.3.0->pyvis) (4.3.3)\nRequirement already satisfied, skipping upgrade: pexpect; sys_platform != \"win32\" in /opt/conda/lib/python3.6/site-packages (from ipython>=5.3.0->pyvis) (4.8.0)\nRequirement already satisfied, skipping upgrade: backcall in /opt/conda/lib/python3.6/site-packages (from ipython>=5.3.0->pyvis) (0.1.0)\nRequirement already satisfied, skipping upgrade: pickleshare in /opt/conda/lib/python3.6/site-packages (from ipython>=5.3.0->pyvis) (0.7.5)\nRequirement already satisfied, skipping upgrade: pygments in /opt/conda/lib/python3.6/site-packages (from ipython>=5.3.0->pyvis) (2.5.2)\nRequirement already satisfied, skipping upgrade: prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0 in /opt/conda/lib/python3.6/site-packages (from ipython>=5.3.0->pyvis) (3.0.3)\nRequirement already satisfied, skipping upgrade: setuptools>=18.5 in /opt/conda/lib/python3.6/site-packages (from ipython>=5.3.0->pyvis) (45.2.0.post20200210)\nRequirement already satisfied, skipping upgrade: jedi>=0.10 in /opt/conda/lib/python3.6/site-packages (from ipython>=5.3.0->pyvis) (0.16.0)\nRequirement already satisfied, skipping upgrade: MarkupSafe>=0.23 in /opt/conda/lib/python3.6/site-packages (from jinja2>=2.9.6->pyvis) (1.1.1)\nRequirement already satisfied, skipping upgrade: ipython-genutils in /opt/conda/lib/python3.6/site-packages (from traitlets>=4.2->ipython>=5.3.0->pyvis) (0.2.0)\nRequirement already satisfied, skipping upgrade: six in /opt/conda/lib/python3.6/site-packages (from traitlets>=4.2->ipython>=5.3.0->pyvis) (1.14.0)\nRequirement already satisfied, skipping upgrade: ptyprocess>=0.5 in /opt/conda/lib/python3.6/site-packages (from pexpect; sys_platform != \"win32\"->ipython>=5.3.0->pyvis) (0.6.0)\nRequirement already satisfied, skipping upgrade: wcwidth in /opt/conda/lib/python3.6/site-packages (from prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0->ipython>=5.3.0->pyvis) (0.1.8)\nRequirement already satisfied, skipping upgrade: parso>=0.5.2 in /opt/conda/lib/python3.6/site-packages (from jedi>=0.10->ipython>=5.3.0->pyvis) (0.6.1)\nCollecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_craft_md-0.2.4.tar.gz\n  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_craft_md-0.2.4.tar.gz (70.1 MB)\n\u001b[K     |████████████████████████████████| 70.1 MB 80 kB/s  eta 0:00:01\n\u001b[?25hRequirement already satisfied: spacy>=2.2.1 in /opt/conda/lib/python3.6/site-packages (from en-ner-craft-md==0.2.4) (2.2.3)\nRequirement already satisfied: preshed<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (3.0.2)\nRequirement already satisfied: catalogue<1.1.0,>=0.0.7 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (1.0.0)\nRequirement already satisfied: plac<1.2.0,>=0.9.6 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (0.9.6)\nRequirement already satisfied: requests<3.0.0,>=2.13.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (2.22.0)\nRequirement already satisfied: thinc<7.4.0,>=7.3.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (7.3.1)\nRequirement already satisfied: wasabi<1.1.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (0.6.0)\nRequirement already satisfied: srsly<1.1.0,>=0.1.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (1.0.2)\nRequirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (1.0.2)\nRequirement already satisfied: cymem<2.1.0,>=2.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (2.0.3)\nRequirement already satisfied: numpy>=1.15.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (1.18.1)\nRequirement already satisfied: setuptools in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (45.2.0.post20200210)\n","name":"stdout"},{"output_type":"stream","text":"Requirement already satisfied: blis<0.5.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (0.4.1)\nRequirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /opt/conda/lib/python3.6/site-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-craft-md==0.2.4) (1.5.0)\nRequirement already satisfied: idna<2.9,>=2.5 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-craft-md==0.2.4) (2.8)\nRequirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-craft-md==0.2.4) (1.24.3)\nRequirement already satisfied: chardet<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-craft-md==0.2.4) (3.0.4)\nRequirement already satisfied: certifi>=2017.4.17 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-craft-md==0.2.4) (2019.11.28)\nRequirement already satisfied: tqdm<5.0.0,>=4.10.0 in /opt/conda/lib/python3.6/site-packages (from thinc<7.4.0,>=7.3.0->spacy>=2.2.1->en-ner-craft-md==0.2.4) (4.42.0)\nRequirement already satisfied: zipp>=0.5 in /opt/conda/lib/python3.6/site-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-craft-md==0.2.4) (2.2.0)\nBuilding wheels for collected packages: en-ner-craft-md\n  Building wheel for en-ner-craft-md (setup.py) ... \u001b[?25ldone\n\u001b[?25h  Created wheel for en-ner-craft-md: filename=en_ner_craft_md-0.2.4-py3-none-any.whl size=70540605 sha256=466b96f774a054fec6c5fcec5959dbf877ed14ab68282e8f5723d2592dc0f4ab\n  Stored in directory: /root/.cache/pip/wheels/64/83/5e/e01c4ac4512b393b6f5a0a1362b5bf1cc39cf5e3a13e9d32db\nSuccessfully built en-ner-craft-md\nInstalling collected packages: en-ner-craft-md\nSuccessfully installed en-ner-craft-md-0.2.4\nCollecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_jnlpba_md-0.2.4.tar.gz\n  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_jnlpba_md-0.2.4.tar.gz (70.1 MB)\n\u001b[K     |████████████████████████████████| 70.1 MB 111 kB/s eta 0:00:01\n\u001b[?25hRequirement already satisfied: spacy>=2.2.1 in /opt/conda/lib/python3.6/site-packages (from en-ner-jnlpba-md==0.2.4) (2.2.3)\nRequirement already satisfied: setuptools in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (45.2.0.post20200210)\nRequirement already satisfied: srsly<1.1.0,>=0.1.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (1.0.2)\nRequirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (1.0.2)\nRequirement already satisfied: catalogue<1.1.0,>=0.0.7 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (1.0.0)\nRequirement already satisfied: preshed<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (3.0.2)\nRequirement already satisfied: numpy>=1.15.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (1.18.1)\nRequirement already satisfied: cymem<2.1.0,>=2.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (2.0.3)\nRequirement already satisfied: requests<3.0.0,>=2.13.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (2.22.0)\nRequirement already satisfied: wasabi<1.1.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (0.6.0)\nRequirement already satisfied: plac<1.2.0,>=0.9.6 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (0.9.6)\nRequirement already satisfied: blis<0.5.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (0.4.1)\nRequirement already satisfied: thinc<7.4.0,>=7.3.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (7.3.1)\nRequirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /opt/conda/lib/python3.6/site-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (1.5.0)\nRequirement already satisfied: chardet<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (3.0.4)\nRequirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (1.24.3)\nRequirement already satisfied: idna<2.9,>=2.5 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (2.8)\nRequirement already satisfied: certifi>=2017.4.17 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (2019.11.28)\nRequirement already satisfied: tqdm<5.0.0,>=4.10.0 in /opt/conda/lib/python3.6/site-packages (from thinc<7.4.0,>=7.3.0->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (4.42.0)\nRequirement already satisfied: zipp>=0.5 in /opt/conda/lib/python3.6/site-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (2.2.0)\nBuilding wheels for collected packages: en-ner-jnlpba-md\n  Building wheel for en-ner-jnlpba-md (setup.py) ... \u001b[?25ldone\n\u001b[?25h  Created wheel for en-ner-jnlpba-md: filename=en_ner_jnlpba_md-0.2.4-py3-none-any.whl size=70531494 sha256=4f34004775bebefc4ea911a649b5144e67b52900c37b2717e0e9cd6aa1faabdd\n  Stored in directory: /root/.cache/pip/wheels/4e/5a/99/0391f161e42ad59b820878ca4e495bfc175cb9ba6ffbc3a229\nSuccessfully built en-ner-jnlpba-md\nInstalling collected packages: en-ner-jnlpba-md\nSuccessfully installed en-ner-jnlpba-md-0.2.4\nCollecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_bc5cdr_md-0.2.4.tar.gz\n  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_bc5cdr_md-0.2.4.tar.gz (70.1 MB)\n\u001b[K     |████████████████████████████████| 70.1 MB 106 kB/s eta 0:00:01\n\u001b[?25hRequirement already satisfied: spacy>=2.2.1 in /opt/conda/lib/python3.6/site-packages (from en-ner-bc5cdr-md==0.2.4) (2.2.3)\nRequirement already satisfied: preshed<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (3.0.2)\nRequirement already satisfied: requests<3.0.0,>=2.13.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (2.22.0)\nRequirement already satisfied: setuptools in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (45.2.0.post20200210)\nRequirement already satisfied: plac<1.2.0,>=0.9.6 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (0.9.6)\nRequirement already satisfied: wasabi<1.1.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (0.6.0)\nRequirement already satisfied: catalogue<1.1.0,>=0.0.7 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (1.0.0)\nRequirement already satisfied: blis<0.5.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (0.4.1)\nRequirement already satisfied: cymem<2.1.0,>=2.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (2.0.3)\nRequirement already satisfied: numpy>=1.15.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (1.18.1)\nRequirement already satisfied: thinc<7.4.0,>=7.3.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (7.3.1)\nRequirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (1.0.2)\nRequirement already satisfied: srsly<1.1.0,>=0.1.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (1.0.2)\nRequirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (1.24.3)\n","name":"stdout"},{"output_type":"stream","text":"Requirement already satisfied: certifi>=2017.4.17 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (2019.11.28)\nRequirement already satisfied: chardet<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (3.0.4)\nRequirement already satisfied: idna<2.9,>=2.5 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (2.8)\nRequirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /opt/conda/lib/python3.6/site-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (1.5.0)\nRequirement already satisfied: tqdm<5.0.0,>=4.10.0 in /opt/conda/lib/python3.6/site-packages (from thinc<7.4.0,>=7.3.0->spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (4.42.0)\nRequirement already satisfied: zipp>=0.5 in /opt/conda/lib/python3.6/site-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-bc5cdr-md==0.2.4) (2.2.0)\nBuilding wheels for collected packages: en-ner-bc5cdr-md\n  Building wheel for en-ner-bc5cdr-md (setup.py) ... \u001b[?25ldone\n\u001b[?25h  Created wheel for en-ner-bc5cdr-md: filename=en_ner_bc5cdr_md-0.2.4-py3-none-any.whl size=70531464 sha256=ab2cf389620a59d531f83903677c7b33f3c78402e5c528cf2a6d70dfe97606d9\n  Stored in directory: /root/.cache/pip/wheels/ef/46/ab/a7c167cf18e10044e58be349e252c93e07eec2760d8c1d93ef\nSuccessfully built en-ner-bc5cdr-md\nInstalling collected packages: en-ner-bc5cdr-md\nSuccessfully installed en-ner-bc5cdr-md-0.2.4\nCollecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_bionlp13cg_md-0.2.4.tar.gz\n  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_bionlp13cg_md-0.2.4.tar.gz (70.1 MB)\n\u001b[K     |████████████████████████████████| 70.1 MB 889 kB/s eta 0:00:01\n\u001b[?25hRequirement already satisfied: spacy>=2.2.1 in /opt/conda/lib/python3.6/site-packages (from en-ner-bionlp13cg-md==0.2.4) (2.2.3)\nRequirement already satisfied: requests<3.0.0,>=2.13.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (2.22.0)\nRequirement already satisfied: blis<0.5.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (0.4.1)\nRequirement already satisfied: plac<1.2.0,>=0.9.6 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (0.9.6)\nRequirement already satisfied: setuptools in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (45.2.0.post20200210)\nRequirement already satisfied: srsly<1.1.0,>=0.1.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (1.0.2)\nRequirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (1.0.2)\nRequirement already satisfied: thinc<7.4.0,>=7.3.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (7.3.1)\nRequirement already satisfied: wasabi<1.1.0,>=0.4.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (0.6.0)\nRequirement already satisfied: preshed<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (3.0.2)\nRequirement already satisfied: numpy>=1.15.0 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (1.18.1)\nRequirement already satisfied: cymem<2.1.0,>=2.0.2 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (2.0.3)\nRequirement already satisfied: catalogue<1.1.0,>=0.0.7 in /opt/conda/lib/python3.6/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (1.0.0)\nRequirement already satisfied: certifi>=2017.4.17 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (2019.11.28)\nRequirement already satisfied: chardet<3.1.0,>=3.0.2 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (3.0.4)\nRequirement already satisfied: idna<2.9,>=2.5 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (2.8)\nRequirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /opt/conda/lib/python3.6/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (1.24.3)\nRequirement already satisfied: tqdm<5.0.0,>=4.10.0 in /opt/conda/lib/python3.6/site-packages (from thinc<7.4.0,>=7.3.0->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (4.42.0)\nRequirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /opt/conda/lib/python3.6/site-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (1.5.0)\nRequirement already satisfied: zipp>=0.5 in /opt/conda/lib/python3.6/site-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (2.2.0)\nBuilding wheels for collected packages: en-ner-bionlp13cg-md\n  Building wheel for en-ner-bionlp13cg-md (setup.py) ... \u001b[?25ldone\n\u001b[?25h  Created wheel for en-ner-bionlp13cg-md: filename=en_ner_bionlp13cg_md-0.2.4-py3-none-any.whl size=70542690 sha256=92324042cb3d8cc3695c7ff405881efbe8f44d9a25fd79e5f9902a940d27b01b\n  Stored in directory: /root/.cache/pip/wheels/d8/d6/fe/377a4530f8e2979b48ef8338e64579c7277a85e3647f51f282\nSuccessfully built en-ner-bionlp13cg-md\nInstalling collected packages: en-ner-bionlp13cg-md\nSuccessfully installed en-ner-bionlp13cg-md-0.2.4\n","name":"stdout"}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"!wget -O solr-8.5.0.zip \"https://archive.apache.org/dist/lucene/solr/8.5.0/solr-8.5.0.zip\";\n!unzip solr-8.5.0.zip","execution_count":3,"outputs":[{"output_type":"stream","text":"wget: /opt/conda/lib/libuuid.so.1: no version information available (required by wget)\n--2020-04-08 08:56:29--  https://archive.apache.org/dist/lucene/solr/8.5.0/solr-8.5.0.zip\nResolving archive.apache.org (archive.apache.org)... 163.172.17.199\nConnecting to archive.apache.org (archive.apache.org)|163.172.17.199|:443... connected.\nHTTP request sent, awaiting response... 200 OK\nLength: 191690410 (183M) [application/zip]\nSaving to: ‘solr-8.5.0.zip’\n\nsolr-8.5.0.zip      100%[===================>] 182.81M  66.9MB/s    in 2.7s    \n\n2020-04-08 08:56:32 (66.9 MB/s) - ‘solr-8.5.0.zip’ saved [191690410/191690410]\n\nArchive:  solr-8.5.0.zip\n   creating: solr-8.5.0/\n   creating: solr-8.5.0/contrib/\n   creating: solr-8.5.0/contrib/analysis-extras/\n   creating: solr-8.5.0/contrib/analysis-extras/lib/\n   creating: solr-8.5.0/contrib/analysis-extras/lucene-libs/\n   creating: solr-8.5.0/contrib/clustering/\n   creating: solr-8.5.0/contrib/clustering/lib/\n   creating: solr-8.5.0/contrib/dataimporthandler/\n   creating: solr-8.5.0/contrib/dataimporthandler-extras/\n   creating: solr-8.5.0/contrib/dataimporthandler-extras/lib/\n   creating: solr-8.5.0/contrib/extraction/\n   creating: solr-8.5.0/contrib/extraction/lib/\n   creating: solr-8.5.0/contrib/jaegertracer-configurator/\n   creating: solr-8.5.0/contrib/jaegertracer-configurator/lib/\n   creating: solr-8.5.0/contrib/langid/\n   creating: solr-8.5.0/contrib/langid/lib/\n   creating: solr-8.5.0/contrib/ltr/\n   creating: solr-8.5.0/contrib/prometheus-exporter/\n   creating: solr-8.5.0/contrib/prometheus-exporter/conf/\n   creating: solr-8.5.0/contrib/prometheus-exporter/lib/\n   creating: solr-8.5.0/contrib/prometheus-exporter/lucene-libs/\n   creating: solr-8.5.0/contrib/velocity/\n   creating: solr-8.5.0/contrib/velocity/lib/\n   creating: solr-8.5.0/dist/\n   creating: solr-8.5.0/dist/solrj-lib/\n   creating: solr-8.5.0/dist/test-framework/\n   creating: solr-8.5.0/dist/test-framework/lib/\n   creating: solr-8.5.0/dist/test-framework/lucene-libs/\n   creating: solr-8.5.0/docs/\n   creating: solr-8.5.0/docs/images/\n   creating: solr-8.5.0/example/\n   creating: solr-8.5.0/example/example-DIH/\n   creating: solr-8.5.0/example/example-DIH/hsqldb/\n   creating: solr-8.5.0/example/example-DIH/solr/\n   creating: solr-8.5.0/example/example-DIH/solr/atom/\n   creating: solr-8.5.0/example/example-DIH/solr/atom/conf/\n   creating: solr-8.5.0/example/example-DIH/solr/atom/conf/lang/\n   creating: solr-8.5.0/example/example-DIH/solr/db/\n   creating: solr-8.5.0/example/example-DIH/solr/db/conf/\n   creating: solr-8.5.0/example/example-DIH/solr/db/conf/clustering/\n   creating: solr-8.5.0/example/example-DIH/solr/db/conf/clustering/carrot2/\n   creating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/\n   creating: solr-8.5.0/example/example-DIH/solr/db/conf/xslt/\n   creating: solr-8.5.0/example/example-DIH/solr/db/lib/\n   creating: solr-8.5.0/example/example-DIH/solr/mail/\n   creating: solr-8.5.0/example/example-DIH/solr/mail/conf/\n   creating: solr-8.5.0/example/example-DIH/solr/mail/conf/clustering/\n   creating: solr-8.5.0/example/example-DIH/solr/mail/conf/clustering/carrot2/\n   creating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/\n   creating: solr-8.5.0/example/example-DIH/solr/mail/conf/xslt/\n   creating: solr-8.5.0/example/example-DIH/solr/solr/\n   creating: solr-8.5.0/example/example-DIH/solr/solr/conf/\n   creating: solr-8.5.0/example/example-DIH/solr/solr/conf/clustering/\n   creating: solr-8.5.0/example/example-DIH/solr/solr/conf/clustering/carrot2/\n   creating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/\n   creating: solr-8.5.0/example/example-DIH/solr/solr/conf/xslt/\n   creating: solr-8.5.0/example/example-DIH/solr/tika/\n   creating: solr-8.5.0/example/example-DIH/solr/tika/conf/\n   creating: solr-8.5.0/example/exampledocs/\n   creating: solr-8.5.0/example/files/\n   creating: solr-8.5.0/example/files/browse-resources/\n   creating: solr-8.5.0/example/files/browse-resources/velocity/\n   creating: solr-8.5.0/example/files/conf/\n   creating: solr-8.5.0/example/files/conf/lang/\n   creating: solr-8.5.0/example/files/conf/velocity/\n   creating: solr-8.5.0/example/files/conf/velocity/img/\n   creating: solr-8.5.0/example/files/conf/velocity/js/\n   creating: solr-8.5.0/example/films/\n   creating: solr-8.5.0/licenses/\n   creating: solr-8.5.0/server/\n   creating: solr-8.5.0/server/contexts/\n   creating: solr-8.5.0/server/etc/\n   creating: solr-8.5.0/server/lib/\n   creating: solr-8.5.0/server/lib/ext/\n   creating: solr-8.5.0/server/modules/\n   creating: solr-8.5.0/server/resources/\n   creating: solr-8.5.0/server/scripts/\n   creating: solr-8.5.0/server/scripts/cloud-scripts/\n   creating: solr-8.5.0/server/solr/\n   creating: solr-8.5.0/server/solr-webapp/\n   creating: solr-8.5.0/server/solr-webapp/webapp/\n   creating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/\n   creating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/\n   creating: solr-8.5.0/server/solr-webapp/webapp/css/\n   creating: solr-8.5.0/server/solr-webapp/webapp/css/angular/\n   creating: solr-8.5.0/server/solr-webapp/webapp/img/\n   creating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/\n   creating: solr-8.5.0/server/solr-webapp/webapp/img/ico/\n   creating: solr-8.5.0/server/solr-webapp/webapp/img/jstree/\n   creating: solr-8.5.0/server/solr-webapp/webapp/js/\n   creating: solr-8.5.0/server/solr-webapp/webapp/js/angular/\n   creating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/\n   creating: solr-8.5.0/server/solr-webapp/webapp/libs/\n   creating: solr-8.5.0/server/solr-webapp/webapp/partials/\n   creating: solr-8.5.0/server/solr/configsets/\n   creating: solr-8.5.0/server/solr/configsets/_default/\n   creating: solr-8.5.0/server/solr/configsets/_default/conf/\n   creating: solr-8.5.0/server/solr/configsets/_default/conf/lang/\n   creating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/\n   creating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/\n   creating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/clustering/\n   creating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/clustering/carrot2/\n   creating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/\n   creating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/\n   creating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/xslt/\n  inflating: solr-8.5.0/CHANGES.txt  \n  inflating: solr-8.5.0/LICENSE.txt  \n  inflating: solr-8.5.0/LUCENE_CHANGES.txt  \n  inflating: solr-8.5.0/NOTICE.txt   \n  inflating: solr-8.5.0/README.txt   \n  inflating: solr-8.5.0/contrib/analysis-extras/README.txt  \n  inflating: solr-8.5.0/contrib/analysis-extras/lib/icu4j-62.1.jar  \n  inflating: solr-8.5.0/contrib/analysis-extras/lib/morfologik-fsa-2.1.5.jar  \n  inflating: solr-8.5.0/contrib/analysis-extras/lib/morfologik-polish-2.1.5.jar  \n  inflating: solr-8.5.0/contrib/analysis-extras/lib/morfologik-stemming-2.1.5.jar  \n  inflating: solr-8.5.0/contrib/analysis-extras/lib/morfologik-ukrainian-search-3.9.0.jar  \n  inflating: solr-8.5.0/contrib/analysis-extras/lib/opennlp-tools-1.9.1.jar  \n  inflating: solr-8.5.0/contrib/analysis-extras/lucene-libs/lucene-analyzers-icu-8.5.0.jar  \n  inflating: solr-8.5.0/contrib/analysis-extras/lucene-libs/lucene-analyzers-morfologik-8.5.0.jar  \n  inflating: solr-8.5.0/contrib/analysis-extras/lucene-libs/lucene-analyzers-opennlp-8.5.0.jar  \n  inflating: solr-8.5.0/contrib/analysis-extras/lucene-libs/lucene-analyzers-smartcn-8.5.0.jar  \n  inflating: solr-8.5.0/contrib/analysis-extras/lucene-libs/lucene-analyzers-stempel-8.5.0.jar  \n  inflating: solr-8.5.0/contrib/clustering/README.txt  \n  inflating: solr-8.5.0/contrib/clustering/lib/attributes-binder-1.3.3.jar  \n  inflating: solr-8.5.0/contrib/clustering/lib/carrot2-guava-18.0.jar  \n  inflating: solr-8.5.0/contrib/clustering/lib/carrot2-mini-3.16.0.jar  \n  inflating: solr-8.5.0/contrib/clustering/lib/jackson-annotations-2.10.1.jar  \n  inflating: solr-8.5.0/contrib/clustering/lib/jackson-databind-2.10.1.jar  \n  inflating: solr-8.5.0/contrib/clustering/lib/simple-xml-safe-2.7.1.jar  \n  inflating: solr-8.5.0/contrib/dataimporthandler-extras/lib/activation-1.1.1.jar  \n  inflating: solr-8.5.0/contrib/dataimporthandler-extras/lib/gimap-1.5.1.jar  \n  inflating: solr-8.5.0/contrib/dataimporthandler-extras/lib/javax.mail-1.5.1.jar  \n  inflating: solr-8.5.0/contrib/dataimporthandler/README.txt  \n  inflating: solr-8.5.0/contrib/extraction/README.txt  \n  inflating: solr-8.5.0/contrib/extraction/lib/apache-mime4j-core-0.8.3.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/apache-mime4j-dom-0.8.3.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/aspectjrt-1.8.0.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/bcmail-jdk15on-1.64.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/bcpkix-jdk15on-1.64.jar  \n","name":"stdout"},{"output_type":"stream","text":"  inflating: solr-8.5.0/contrib/extraction/lib/bcprov-jdk15on-1.64.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/boilerpipe-1.1.0.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/commons-collections4-4.2.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/commons-compress-1.19.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/commons-csv-1.7.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/curvesapi-1.06.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/dec-0.1.2.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/fontbox-2.0.17.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/icu4j-62.1.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/isoparser-1.1.22.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/jackcess-3.0.1.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/jackcess-encrypt-3.0.0.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/java-libpst-0.8.1.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/jdom2-2.0.6.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/jempbox-1.8.16.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/jmatio-1.5.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/juniversalchardet-1.0.3.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/metadata-extractor-2.11.0.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/parso-2.0.11.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/pdfbox-2.0.17.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/pdfbox-tools-2.0.17.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/poi-4.1.1.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/poi-ooxml-4.1.1.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/poi-ooxml-schemas-4.1.1.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/poi-scratchpad-4.1.1.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/rome-1.12.2.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/rome-utils-1.12.2.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/tagsoup-1.2.1.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/tika-core-1.23.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/tika-java7-1.23.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/tika-parsers-1.23.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/tika-xmp-1.23.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/vorbis-java-core-0.8.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/vorbis-java-tika-0.8.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/xercesImpl-2.12.0.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/xml-apis-1.4.01.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/xmlbeans-3.1.0.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/xmpcore-5.1.3.jar  \n  inflating: solr-8.5.0/contrib/extraction/lib/xz-1.8.jar  \n  inflating: solr-8.5.0/contrib/jaegertracer-configurator/README.txt  \n  inflating: solr-8.5.0/contrib/jaegertracer-configurator/lib/jaeger-core-1.1.0.jar  \n  inflating: solr-8.5.0/contrib/jaegertracer-configurator/lib/jaeger-thrift-1.1.0.jar  \n  inflating: solr-8.5.0/contrib/jaegertracer-configurator/lib/libthrift-0.13.0.jar  \n  inflating: solr-8.5.0/contrib/langid/README.txt  \n  inflating: solr-8.5.0/contrib/langid/lib/jsonic-1.2.7.jar  \n  inflating: solr-8.5.0/contrib/langid/lib/langdetect-1.1-20120112.jar  \n  inflating: solr-8.5.0/contrib/langid/lib/opennlp-tools-1.9.1.jar  \n  inflating: solr-8.5.0/contrib/ltr/README.txt  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/README.txt  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/conf/grafana-solr-dashboard.json  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/conf/solr-exporter-config.xml  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/argparse4j-0.8.1.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/jackson-annotations-2.10.1.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/jackson-core-2.10.1.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/jackson-databind-2.10.1.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/jackson-jq-0.0.8.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/log4j-api-2.11.2.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/log4j-core-2.11.2.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/log4j-slf4j-impl-2.11.2.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/simpleclient-0.2.0.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/simpleclient_common-0.2.0.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/simpleclient_httpserver-0.2.0.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lib/slf4j-api-1.7.24.jar  \n  inflating: solr-8.5.0/contrib/prometheus-exporter/lucene-libs/lucene-analyzers-common-8.5.0.jar  \n  inflating: solr-8.5.0/contrib/velocity/lib/commons-lang3-3.9.jar  \n  inflating: solr-8.5.0/contrib/velocity/lib/velocity-engine-core-2.0.jar  \n  inflating: solr-8.5.0/contrib/velocity/lib/velocity-tools-generic-3.0.jar  \n  inflating: solr-8.5.0/contrib/velocity/lib/velocity-tools-view-3.0.jar  \n  inflating: solr-8.5.0/contrib/velocity/lib/velocity-tools-view-jsp-3.0.jar  \n  inflating: solr-8.5.0/dist/solr-analysis-extras-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-analytics-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-cell-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-clustering-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-core-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-dataimporthandler-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-dataimporthandler-extras-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-jaegertracer-configurator-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-langid-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-ltr-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-prometheus-exporter-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-solrj-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-test-framework-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solr-velocity-8.5.0.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/commons-io-2.6.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/commons-math3-3.6.1.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/http2-client-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/http2-common-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/http2-hpack-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/http2-http-client-transport-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/httpclient-4.5.10.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/httpcore-4.4.12.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/httpmime-4.5.10.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/jcl-over-slf4j-1.7.24.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/jetty-alpn-client-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/jetty-alpn-java-client-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/jetty-client-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/jetty-http-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/jetty-io-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/jetty-util-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/netty-buffer-4.1.29.Final.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/netty-codec-4.1.29.Final.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/netty-common-4.1.29.Final.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/netty-handler-4.1.29.Final.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/netty-resolver-4.1.29.Final.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/netty-transport-4.1.29.Final.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/netty-transport-native-epoll-4.1.29.Final.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/netty-transport-native-unix-common-4.1.29.Final.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/slf4j-api-1.7.24.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/stax2-api-3.1.4.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/woodstox-core-asl-4.4.1.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/zookeeper-3.5.5.jar  \n  inflating: solr-8.5.0/dist/solrj-lib/zookeeper-jute-3.5.5.jar  \n  inflating: solr-8.5.0/dist/test-framework/README.txt  \n  inflating: solr-8.5.0/dist/test-framework/lib/ant-1.8.2.jar  \n  inflating: solr-8.5.0/dist/test-framework/lib/hamcrest-core-1.3.jar  \n  inflating: solr-8.5.0/dist/test-framework/lib/junit-4.12.jar  \n  inflating: solr-8.5.0/dist/test-framework/lib/junit4-ant-2.7.2.jar  \n  inflating: solr-8.5.0/dist/test-framework/lib/opentracing-mock-0.33.0.jar  \n  inflating: solr-8.5.0/dist/test-framework/lib/randomizedtesting-runner-2.7.2.jar  \n","name":"stdout"},{"output_type":"stream","text":"  inflating: solr-8.5.0/dist/test-framework/lucene-libs/lucene-test-framework-8.5.0.jar  \n  inflating: solr-8.5.0/docs/images/solr.svg  \n  inflating: solr-8.5.0/docs/index.html  \n  inflating: solr-8.5.0/example/README.txt  \n  inflating: solr-8.5.0/example/example-DIH/README.txt  \n  inflating: solr-8.5.0/example/example-DIH/hsqldb/ex.script  \n  inflating: solr-8.5.0/example/example-DIH/solr/atom/conf/atom-data-config.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/atom/conf/lang/stopwords_en.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/atom/conf/managed-schema  \n  inflating: solr-8.5.0/example/example-DIH/solr/atom/conf/protwords.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/atom/conf/solrconfig.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/atom/conf/synonyms.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/atom/conf/url_types.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/atom/core.properties  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/clustering/carrot2/kmeans-attributes.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/clustering/carrot2/lingo-attributes.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/clustering/carrot2/stc-attributes.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/currency.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/db-data-config.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/elevate.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/contractions_ca.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/contractions_fr.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/contractions_ga.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/contractions_it.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/hyphenations_ga.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stemdict_nl.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stoptags_ja.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_ar.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_bg.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_ca.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_ckb.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_cz.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_da.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_de.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_el.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_en.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_es.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_eu.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_fa.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_fi.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_fr.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_ga.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_gl.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_hi.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_hu.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_hy.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_id.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_it.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_ja.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_lv.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_nl.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_no.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_pt.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_ro.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_ru.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_sv.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_th.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/stopwords_tr.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/lang/userdict_ja.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/managed-schema  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/mapping-FoldToASCII.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/mapping-ISOLatin1Accent.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/protwords.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/solrconfig.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/spellings.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/stopwords.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/synonyms.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/update-script.js  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/xslt/example.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/xslt/example_atom.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/xslt/example_rss.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/xslt/luke.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/conf/xslt/updateXml.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/core.properties  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/lib/derby-10.9.1.0.jar  \n  inflating: solr-8.5.0/example/example-DIH/solr/db/lib/hsqldb-2.4.0.jar  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/clustering/carrot2/kmeans-attributes.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/clustering/carrot2/lingo-attributes.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/clustering/carrot2/stc-attributes.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/currency.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/elevate.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/contractions_ca.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/contractions_fr.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/contractions_ga.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/contractions_it.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/hyphenations_ga.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stemdict_nl.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stoptags_ja.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_ar.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_bg.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_ca.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_ckb.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_cz.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_da.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_de.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_el.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_en.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_es.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_eu.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_fa.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_fi.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_fr.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_ga.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_gl.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_hi.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_hu.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_hy.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_id.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_it.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_ja.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_lv.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_nl.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_no.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_pt.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_ro.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_ru.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_sv.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_th.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/stopwords_tr.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/lang/userdict_ja.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/mail-data-config.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/managed-schema  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/mapping-FoldToASCII.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/mapping-ISOLatin1Accent.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/protwords.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/solrconfig.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/spellings.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/stopwords.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/synonyms.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/update-script.js  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/xslt/example.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/xslt/example_atom.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/xslt/example_rss.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/xslt/luke.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/conf/xslt/updateXml.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/mail/core.properties  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/clustering/carrot2/kmeans-attributes.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/clustering/carrot2/lingo-attributes.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/clustering/carrot2/stc-attributes.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/currency.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/elevate.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/contractions_ca.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/contractions_fr.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/contractions_ga.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/contractions_it.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/hyphenations_ga.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stemdict_nl.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stoptags_ja.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_ar.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_bg.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_ca.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_ckb.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_cz.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_da.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_de.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_el.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_en.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_es.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_eu.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_fa.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_fi.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_fr.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_ga.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_gl.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_hi.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_hu.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_hy.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_id.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_it.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_ja.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_lv.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_nl.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_no.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_pt.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_ro.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_ru.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_sv.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_th.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/stopwords_tr.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/lang/userdict_ja.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/managed-schema  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/mapping-FoldToASCII.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/mapping-ISOLatin1Accent.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/protwords.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/solr-data-config.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/solrconfig.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/spellings.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/stopwords.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/synonyms.txt  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/update-script.js  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/xslt/example.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/xslt/example_atom.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/xslt/example_rss.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/xslt/luke.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/conf/xslt/updateXml.xsl  \n  inflating: solr-8.5.0/example/example-DIH/solr/solr/core.properties  \n  inflating: solr-8.5.0/example/example-DIH/solr/tika/conf/managed-schema  \n  inflating: solr-8.5.0/example/example-DIH/solr/tika/conf/solrconfig.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/tika/conf/tika-data-config.xml  \n  inflating: solr-8.5.0/example/example-DIH/solr/tika/core.properties  \n  inflating: solr-8.5.0/example/exampledocs/books.csv  \n  inflating: solr-8.5.0/example/exampledocs/books.json  \n  inflating: solr-8.5.0/example/exampledocs/gb18030-example.xml  \n  inflating: solr-8.5.0/example/exampledocs/hd.xml  \n  inflating: solr-8.5.0/example/exampledocs/ipod_other.xml  \n  inflating: solr-8.5.0/example/exampledocs/ipod_video.xml  \n  inflating: solr-8.5.0/example/exampledocs/manufacturers.xml  \n  inflating: solr-8.5.0/example/exampledocs/mem.xml  \n  inflating: solr-8.5.0/example/exampledocs/money.xml  \n  inflating: solr-8.5.0/example/exampledocs/monitor.xml  \n  inflating: solr-8.5.0/example/exampledocs/monitor2.xml  \n  inflating: solr-8.5.0/example/exampledocs/more_books.jsonl  \n  inflating: solr-8.5.0/example/exampledocs/mp500.xml  \n  inflating: solr-8.5.0/example/exampledocs/post.jar  \n  inflating: solr-8.5.0/example/exampledocs/sample.html  \n  inflating: solr-8.5.0/example/exampledocs/sd500.xml  \n  inflating: solr-8.5.0/example/exampledocs/solr-word.pdf  \n  inflating: solr-8.5.0/example/exampledocs/solr.xml  \n  inflating: solr-8.5.0/example/exampledocs/utf8-example.xml  \n  inflating: solr-8.5.0/example/exampledocs/vidcard.xml  \n  inflating: solr-8.5.0/example/files/README.txt  \n  inflating: solr-8.5.0/example/files/browse-resources/velocity/resources.properties  \n  inflating: solr-8.5.0/example/files/browse-resources/velocity/resources_de_DE.properties  \n  inflating: solr-8.5.0/example/files/browse-resources/velocity/resources_fr_FR.properties  \n  inflating: solr-8.5.0/example/files/conf/currency.xml  \n  inflating: solr-8.5.0/example/files/conf/elevate.xml  \n  inflating: solr-8.5.0/example/files/conf/email_url_types.txt  \n  inflating: solr-8.5.0/example/files/conf/lang/contractions_ca.txt  \n  inflating: solr-8.5.0/example/files/conf/lang/contractions_fr.txt  \n  inflating: solr-8.5.0/example/files/conf/lang/contractions_ga.txt  \n  inflating: solr-8.5.0/example/files/conf/lang/contractions_it.txt  \n  inflating: solr-8.5.0/example/files/conf/lang/hyphenations_ga.txt  \n","name":"stdout"},{"output_type":"stream","text":"  inflating: solr-8.5.0/example/files/conf/lang/stemdict_nl.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stoptags_ja.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_ar.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_bg.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_ca.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_cz.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_da.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_de.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_el.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_en.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_es.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_eu.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_fa.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_fi.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_fr.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_ga.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_gl.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_hi.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_hu.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_hy.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_id.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_it.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_ja.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_lv.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_nl.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_no.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_pt.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_ro.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_ru.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_sv.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_th.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/stopwords_tr.txt  \r\n  inflating: solr-8.5.0/example/files/conf/lang/userdict_ja.txt  \r\n  inflating: solr-8.5.0/example/files/conf/managed-schema  \r\n  inflating: solr-8.5.0/example/files/conf/params.json  \r\n  inflating: solr-8.5.0/example/files/conf/protwords.txt  \r\n  inflating: solr-8.5.0/example/files/conf/solrconfig.xml  \r\n  inflating: solr-8.5.0/example/files/conf/stopwords.txt  \r\n  inflating: solr-8.5.0/example/files/conf/synonyms.txt  \r\n  inflating: solr-8.5.0/example/files/conf/update-script.js  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/browse.vm  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/dropit.js  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/facet_doc_type.vm  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/facet_text_shingles.vm  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/facets.vm  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/footer.vm  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/head.vm  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/hit.vm  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/img/english_640.png  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/img/france_640.png  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/img/germany_640.png  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/img/globe_256.png  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/jquery.tx3-tag-cloud.js  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/js/dropit.js  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/js/jquery.autocomplete.js  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/js/jquery.tx3-tag-cloud.js  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/layout.vm  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/macros.vm  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/mime_type_lists.vm  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/results.vm  \r\n  inflating: solr-8.5.0/example/files/conf/velocity/results_list.vm  \r\n  inflating: solr-8.5.0/example/films/README.txt  \r\n  inflating: solr-8.5.0/example/films/film_data_generator.py  \r\n  inflating: solr-8.5.0/example/films/films-LICENSE.txt  \r\n  inflating: solr-8.5.0/example/films/films.csv  \r\n  inflating: solr-8.5.0/example/films/films.json  \r\n  inflating: solr-8.5.0/example/films/films.xml  \r\n  inflating: solr-8.5.0/licenses/activation-1.1.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/activation-LICENSE-CDDL.txt  \r\n  inflating: solr-8.5.0/licenses/android-json-0.0.20131108.vaadin1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/android-json-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/android-json-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/ant-1.8.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/ant-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/ant-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/antlr4-runtime-4.5.1-1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/antlr4-runtime-LICENSE-BSD.txt  \r\n  inflating: solr-8.5.0/licenses/antlr4-runtime-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/apache-mime4j-core-0.8.3.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/apache-mime4j-core-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/apache-mime4j-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/apache-mime4j-dom-0.8.3.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/apache-mime4j-dom-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/apache-mime4j-dom-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/argparse4j-0.8.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/argparse4j-LICENSE-MIT.txt  \r\n  inflating: solr-8.5.0/licenses/argparse4j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/asciidoctor-ant-1.6.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/asciidoctor-ant-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/asciidoctor-ant-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/asm-7.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/asm-LICENSE-BSD.txt  \r\n  inflating: solr-8.5.0/licenses/asm-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/asm-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/asm-commons-7.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/asm-commons-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/asm-commons-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/aspectjrt-1.8.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/aspectjrt-LICENSE-EPL.txt  \r\n  inflating: solr-8.5.0/licenses/attributes-binder-1.3.3.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/attributes-binder-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/attributes-binder-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/avatica-core-1.13.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/avatica-core-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/avatica-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/bcmail-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/bcmail-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/bcmail-jdk15on-1.64.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/bcpkix-jdk15on-1.64.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/bcpkix-jdk15on-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/bcpkix-jdk15on-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/bcprov-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/bcprov-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/bcprov-jdk15on-1.64.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/boilerpipe-1.1.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/boilerpipe-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/boilerpipe-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/byte-buddy-1.9.3.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/byte-buddy-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/byte-buddy-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/caffeine-2.8.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/caffeine-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/caffeine-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/calcite-core-1.18.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/calcite-core-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/calcite-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/calcite-linq4j-1.18.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/calcite-linq4j-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/calcite-linq4j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/carrot2-guava-18.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/carrot2-guava-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/carrot2-guava-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/carrot2-mini-3.16.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/carrot2-mini-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/carrot2-mini-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-cli-1.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-cli-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-cli-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-codec-1.13.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-codec-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-codec-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-collections-3.2.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-collections-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-collections-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-collections4-4.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-collections4-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-collections4-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-compiler-3.0.9.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-compiler-LICENSE-BSD.txt  \r\n  inflating: solr-8.5.0/licenses/commons-compiler-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-compress-1.19.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-compress-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-compress-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-configuration-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-configuration-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-configuration2-2.1.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-configuration2-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-configuration2-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-csv-1.7.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-csv-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-csv-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-digester-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-digester-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-exec-1.3.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-exec-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-exec-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-fileupload-1.3.3.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-fileupload-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-fileupload-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-io-2.6.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-io-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-io-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-lang3-3.9.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-lang3-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-lang3-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-logging-1.1.3.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-logging-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-logging-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-math3-3.6.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-math3-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-math3-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/commons-text-1.6.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/commons-text-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/commons-text-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/curator-client-2.13.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/curator-client-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/curator-client-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/curator-framework-2.13.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/curator-framework-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/curator-framework-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/curator-recipes-2.13.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/curator-recipes-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/curator-recipes-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/curvesapi-1.06.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/curvesapi-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/curvesapi-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/dec-0.1.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/dec-LICENSE-MIT.txt  \r\n  inflating: solr-8.5.0/licenses/dec-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/derby-10.9.1.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/derby-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/derby-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/disruptor-3.4.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/disruptor-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/disruptor-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/eigenbase-properties-1.1.5.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/eigenbase-properties-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/eigenbase-properties-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/fontbox-2.0.17.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/fontbox-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/fontbox-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/gimap-1.5.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/gimap-LICENSE-CDDL.txt  \r\n  inflating: solr-8.5.0/licenses/guava-25.1-jre.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/guava-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/guava-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-annotations-3.2.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hadoop-annotations-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-annotations-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-auth-3.2.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hadoop-auth-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-auth-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-common-3.2.0-tests.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hadoop-common-3.2.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hadoop-common-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-common-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-common-tests-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-common-tests-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-hdfs-3.2.0-tests.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hadoop-hdfs-3.2.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hadoop-hdfs-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-hdfs-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-hdfs-client-3.2.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hadoop-hdfs-client-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-hdfs-client-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-hdfs-tests-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-hdfs-tests-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-minicluster-3.2.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hadoop-minicluster-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-minicluster-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-minikdc-3.2.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hadoop-minikdc-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/hadoop-minikdc-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hamcrest-core-1.3.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hamcrest-core-LICENSE-BSD.txt  \r\n  inflating: solr-8.5.0/licenses/hamcrest-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hppc-0.8.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hppc-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/hppc-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/hsqldb-2.4.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/hsqldb-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/hsqldb-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/htrace-core4-4.1.0-incubating.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/htrace-core4-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/htrace-core4-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/http2-client-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/http2-client-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/http2-client-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/http2-common-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/http2-common-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/http2-common-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/http2-hpack-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/http2-hpack-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/http2-hpack-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/http2-http-client-transport-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/http2-http-client-transport-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/http2-http-client-transport-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/http2-server-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/http2-server-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/http2-server-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/httpclient-4.5.10.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/httpclient-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/httpclient-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/httpcore-4.4.12.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/httpcore-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/httpcore-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/httpmime-4.5.10.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/httpmime-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/httpmime-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/icu4j-62.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/icu4j-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/icu4j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/isoparser-1.1.22.jar.sha1  \r\n","name":"stdout"},{"output_type":"stream","text":"  inflating: solr-8.5.0/licenses/isoparser-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/isoparser-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jackcess-3.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jackcess-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jackcess-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jackcess-encrypt-3.0.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jackcess-encrypt-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jackcess-encrypt-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-annotations-2.10.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jackson-annotations-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-annotations-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-core-2.10.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jackson-core-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-core-asl-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-core-asl-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-databind-2.10.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jackson-databind-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-databind-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-dataformat-smile-2.10.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jackson-dataformat-smile-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-dataformat-smile-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-jq-0.0.8.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jackson-jq-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-jq-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-mapper-asl-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jackson-mapper-asl-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jaeger-core-1.1.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jaeger-core-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jaeger-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jaeger-thrift-1.1.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jaeger-thrift-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jaeger-thrift-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/janino-3.0.9.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/janino-LICENSE-BSD.txt  \r\n  inflating: solr-8.5.0/licenses/janino-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/java-libpst-0.8.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/java-libpst-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/java-libpst-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/java-semver-0.9.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/java-semver-LICENSE-MIT.txt  \r\n  inflating: solr-8.5.0/licenses/javax.mail-1.5.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/javax.mail-LICENSE-CDDL.txt  \r\n  inflating: solr-8.5.0/licenses/javax.servlet-api-3.1.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/javax.servlet-api-LICENSE-CDDL.txt  \r\n  inflating: solr-8.5.0/licenses/javax.servlet-api-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jcl-over-slf4j-1.7.24.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jcl-over-slf4j-LICENSE-MIT.txt  \r\n  inflating: solr-8.5.0/licenses/jcl-over-slf4j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jdom2-2.0.6.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jdom2-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/jdom2-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jempbox-1.8.16.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jempbox-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jempbox-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jersey-core-1.19.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jersey-core-LICENSE-CDDL.txt  \r\n  inflating: solr-8.5.0/licenses/jersey-server-1.19.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jersey-server-LICENSE-CDDL.txt  \r\n  inflating: solr-8.5.0/licenses/jersey-servlet-1.19.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jersey-servlet-LICENSE-CDDL.txt  \r\n  inflating: solr-8.5.0/licenses/jetty-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jetty-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jetty-alpn-client-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-alpn-java-client-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-alpn-java-server-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-alpn-server-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-client-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-continuation-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-deploy-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-http-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-io-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-jmx-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-rewrite-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-security-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-server-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-servlet-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-servlets-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-util-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-webapp-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jetty-xml-9.4.24.v20191120.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jmatio-1.5.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jmatio-LICENSE-BSD.txt  \r\n  inflating: solr-8.5.0/licenses/jmatio-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jose4j-0.6.5.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jose4j-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jose4j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/json-path-2.4.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/json-path-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/json-path-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jsonic-1.2.7.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jsonic-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/jsonic-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/jsoup-1.12.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jsoup-LICENSE-MIT.txt  \r\n  inflating: solr-8.5.0/licenses/jul-to-slf4j-1.7.24.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/jul-to-slf4j-LICENSE-MIT.txt  \r\n  inflating: solr-8.5.0/licenses/jul-to-slf4j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/junit-4.12.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/junit-LICENSE-CPL.txt  \r\n  inflating: solr-8.5.0/licenses/junit-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/junit4-ant-2.7.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/junit4-ant-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/junit4-ant-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/juniversalchardet-1.0.3.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/juniversalchardet-LICENSE-MPL.txt  \r\n  inflating: solr-8.5.0/licenses/juniversalchardet-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-admin-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerb-admin-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-admin-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-client-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerb-client-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-client-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-common-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerb-common-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-common-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-core-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerb-core-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-crypto-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerb-crypto-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-crypto-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-identity-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerb-identity-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-identity-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-server-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerb-server-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-server-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-simplekdc-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerb-simplekdc-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-simplekdc-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-util-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerb-util-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerb-util-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerby-asn1-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerby-asn1-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerby-asn1-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerby-config-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerby-config-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerby-config-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerby-kdc-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerby-kdc-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerby-kdc-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerby-pkix-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerby-pkix-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerby-pkix-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/kerby-util-1.0.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/kerby-util-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/kerby-util-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/langdetect-1.1-20120112.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/langdetect-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/langdetect-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/libthrift-0.13.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/libthrift-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/libthrift-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/log4j-1.2-api-2.11.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/log4j-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/log4j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/log4j-api-2.11.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/log4j-api-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/log4j-api-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/log4j-core-2.11.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/log4j-core-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/log4j-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/log4j-slf4j-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/log4j-slf4j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/log4j-slf4j-impl-2.11.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/log4j-web-2.11.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/log4j-web-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/log4j-web-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/metadata-extractor-2.11.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/metadata-extractor-LICENSE-PD.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-core-4.1.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/metrics-core-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-graphite-4.1.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/metrics-graphite-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-graphite-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-jetty-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-jetty-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-jetty9-4.1.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/metrics-jmx-4.1.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/metrics-jmx-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-jmx-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-json-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-json-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-jvm-4.1.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/metrics-jvm-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-jvm-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-servlets-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/metrics-servlets-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/mina-core-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/mina-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/mockito-core-2.23.4.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/mockito-core-LICENSE-MIT.txt  \r\n  inflating: solr-8.5.0/licenses/morfologik-fsa-2.1.5.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/morfologik-fsa-LICENSE-BSD.txt  \r\n  inflating: solr-8.5.0/licenses/morfologik-fsa-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/morfologik-polish-2.1.5.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/morfologik-polish-LICENSE-BSD.txt  \r\n  inflating: solr-8.5.0/licenses/morfologik-polish-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/morfologik-stemming-2.1.5.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/morfologik-stemming-LICENSE-BSD.txt  \r\n  inflating: solr-8.5.0/licenses/morfologik-stemming-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/morfologik-ukrainian-search-3.9.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/morfologik-ukrainian-search-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/morfologik-ukrainian-search-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/netty-all-4.1.29.Final.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/netty-all-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/netty-all-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/netty-buffer-4.1.29.Final.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/netty-buffer-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/netty-buffer-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/netty-codec-4.1.29.Final.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/netty-codec-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/netty-codec-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/netty-common-4.1.29.Final.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/netty-common-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/netty-common-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/netty-handler-4.1.29.Final.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/netty-handler-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/netty-handler-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/netty-resolver-4.1.29.Final.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/netty-resolver-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/netty-resolver-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/netty-transport-4.1.29.Final.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/netty-transport-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/netty-transport-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/netty-transport-native-epoll-4.1.29.Final.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/netty-transport-native-epoll-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/netty-transport-native-epoll-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/netty-transport-native-unix-common-4.1.29.Final.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/netty-transport-native-unix-common-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/netty-transport-native-unix-common-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/objenesis-2.6.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/objenesis-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/objenesis-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/opennlp-tools-1.9.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/opennlp-tools-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/opennlp-tools-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/opentracing-api-0.33.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/opentracing-api-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/opentracing-api-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/opentracing-mock-0.33.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/opentracing-mock-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/opentracing-mock-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/opentracing-noop-0.33.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/opentracing-noop-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/opentracing-noop-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/opentracing-util-0.33.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/opentracing-util-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/opentracing-util-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/org.restlet-2.4.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/org.restlet-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/org.restlet-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/org.restlet.ext.servlet-2.4.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/org.restlet.ext.servlet-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/org.restlet.ext.servlet-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/parso-2.0.11.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/parso-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/parso-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/pdfbox-2.0.17.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/pdfbox-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/pdfbox-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/pdfbox-tools-2.0.17.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/pdfbox-tools-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/pdfbox-tools-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/poi-4.1.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/poi-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/poi-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/poi-ooxml-4.1.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/poi-ooxml-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/poi-ooxml-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/poi-ooxml-schemas-4.1.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/poi-ooxml-schemas-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/poi-ooxml-schemas-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/poi-scratchpad-4.1.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/poi-scratchpad-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/poi-scratchpad-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/presto-parser-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/presto-parser-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/protobuf-java-3.11.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/protobuf-java-LICENSE-BSD.txt  \r\n  inflating: solr-8.5.0/licenses/protobuf-java-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/randomizedtesting-runner-2.7.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/randomizedtesting-runner-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/randomizedtesting-runner-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/re2j-1.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/re2j-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/re2j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/rome-1.12.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/rome-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/rome-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/rome-utils-1.12.2.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/rome-utils-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/rome-utils-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/rrd4j-3.5.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/rrd4j-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/rrd4j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/s2-geometry-library-java-1.0.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/s2-geometry-library-java-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/s2-geometry-library-java-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/servlet-api-LICENSE-CDDL.txt  \r\n  inflating: solr-8.5.0/licenses/servlet-api-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/simple-xml-safe-2.7.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/simple-xml-safe-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/simple-xml-safe-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/simpleclient-0.2.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/simpleclient-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/simpleclient-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/simpleclient_common-0.2.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/simpleclient_common-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/simpleclient_common-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/simpleclient_httpserver-0.2.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/simpleclient_httpserver-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/simpleclient_httpserver-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/slf4j-LICENSE-MIT.txt  \r\n  inflating: solr-8.5.0/licenses/slf4j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/slf4j-api-1.7.24.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/slf4j-simple-1.7.24.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/slice-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/slice-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/spatial4j-0.7.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/spatial4j-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/spatial4j-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/start.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/stax2-api-3.1.4.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/stax2-api-LICENSE-BSD.txt  \r\n  inflating: solr-8.5.0/licenses/stax2-api-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/t-digest-3.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/t-digest-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/t-digest-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/tagsoup-1.2.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/tagsoup-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/tagsoup-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/tika-core-1.23.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/tika-core-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/tika-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/tika-java7-1.23.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/tika-java7-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/tika-java7-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/tika-parsers-1.23.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/tika-parsers-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/tika-parsers-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/tika-xmp-1.23.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/tika-xmp-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/tika-xmp-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/velocity-engine-core-2.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/velocity-engine-core-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/velocity-engine-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/velocity-tools-generic-3.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/velocity-tools-generic-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/velocity-tools-generic-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/velocity-tools-view-3.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/velocity-tools-view-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/velocity-tools-view-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/velocity-tools-view-jsp-3.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/velocity-tools-view-jsp-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/velocity-tools-view-jsp-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/vorbis-java-core-0.8.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/vorbis-java-core-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/vorbis-java-core-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/vorbis-java-tika-0.8.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/vorbis-java-tika-LICENSE-BSD_LIKE.txt  \r\n  inflating: solr-8.5.0/licenses/vorbis-java-tika-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/woodstox-core-asl-4.4.1.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/woodstox-core-asl-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/woodstox-core-asl-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/xercesImpl-2.12.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/xercesImpl-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/xercesImpl-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/xml-apis-1.4.01.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/xml-apis-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/xml-apis-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/xmlbeans-3.1.0.jar.sha1  \r\n  inflating: solr-8.5.0/licenses/xmlbeans-LICENSE-ASL.txt  \r\n  inflating: solr-8.5.0/licenses/xmlbeans-NOTICE.txt  \r\n  inflating: solr-8.5.0/licenses/xmpcore-5.1.3.jar.sha1  \r\n","name":"stdout"},{"output_type":"stream","text":"  inflating: solr-8.5.0/licenses/xmpcore-LICENSE-BSD.txt  \n  inflating: solr-8.5.0/licenses/xmpcore-NOTICE.txt  \n  inflating: solr-8.5.0/licenses/xz-1.8.jar.sha1  \n  inflating: solr-8.5.0/licenses/xz-LICENSE-PD.txt  \n  inflating: solr-8.5.0/licenses/xz-NOTICE.txt  \n  inflating: solr-8.5.0/licenses/zookeeper-3.5.5.jar.sha1  \n  inflating: solr-8.5.0/licenses/zookeeper-LICENSE-ASL.txt  \n  inflating: solr-8.5.0/licenses/zookeeper-NOTICE.txt  \n  inflating: solr-8.5.0/licenses/zookeeper-jute-3.5.5.jar.sha1  \n  inflating: solr-8.5.0/licenses/zookeeper-jute-LICENSE-ASL.txt  \n  inflating: solr-8.5.0/licenses/zookeeper-jute-NOTICE.txt  \n  inflating: solr-8.5.0/server/README.txt  \n  inflating: solr-8.5.0/server/contexts/solr-jetty-context.xml  \n  inflating: solr-8.5.0/server/etc/jetty-http.xml  \n  inflating: solr-8.5.0/server/etc/jetty-https.xml  \n  inflating: solr-8.5.0/server/etc/jetty-https8.xml  \n  inflating: solr-8.5.0/server/etc/jetty-requestlog.xml  \n  inflating: solr-8.5.0/server/etc/jetty-ssl.xml  \n  inflating: solr-8.5.0/server/etc/jetty.xml  \n  inflating: solr-8.5.0/server/etc/security.policy  \n  inflating: solr-8.5.0/server/etc/security.properties  \n  inflating: solr-8.5.0/server/etc/webdefault.xml  \n  inflating: solr-8.5.0/server/lib/ext/disruptor-3.4.2.jar  \n  inflating: solr-8.5.0/server/lib/ext/jcl-over-slf4j-1.7.24.jar  \n  inflating: solr-8.5.0/server/lib/ext/jul-to-slf4j-1.7.24.jar  \n  inflating: solr-8.5.0/server/lib/ext/log4j-1.2-api-2.11.2.jar  \n  inflating: solr-8.5.0/server/lib/ext/log4j-api-2.11.2.jar  \n  inflating: solr-8.5.0/server/lib/ext/log4j-core-2.11.2.jar  \n  inflating: solr-8.5.0/server/lib/ext/log4j-slf4j-impl-2.11.2.jar  \n  inflating: solr-8.5.0/server/lib/ext/log4j-web-2.11.2.jar  \n  inflating: solr-8.5.0/server/lib/ext/slf4j-api-1.7.24.jar  \n  inflating: solr-8.5.0/server/lib/http2-common-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/http2-hpack-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/http2-server-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/javax.servlet-api-3.1.0.jar  \n  inflating: solr-8.5.0/server/lib/jetty-alpn-java-server-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-alpn-server-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-continuation-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-deploy-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-http-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-io-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-jmx-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-rewrite-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-security-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-server-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-servlet-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-servlets-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-util-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-webapp-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/jetty-xml-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/lib/metrics-core-4.1.2.jar  \n  inflating: solr-8.5.0/server/lib/metrics-graphite-4.1.2.jar  \n  inflating: solr-8.5.0/server/lib/metrics-jetty9-4.1.2.jar  \n  inflating: solr-8.5.0/server/lib/metrics-jmx-4.1.2.jar  \n  inflating: solr-8.5.0/server/lib/metrics-jvm-4.1.2.jar  \n  inflating: solr-8.5.0/server/modules/http.mod  \n  inflating: solr-8.5.0/server/modules/https.mod  \n  inflating: solr-8.5.0/server/modules/https8.mod  \n  inflating: solr-8.5.0/server/modules/requestlog.mod  \n  inflating: solr-8.5.0/server/modules/server.mod  \n  inflating: solr-8.5.0/server/modules/ssl.mod  \n  inflating: solr-8.5.0/server/resources/jetty-logging.properties  \n  inflating: solr-8.5.0/server/resources/log4j2-console.xml  \n  inflating: solr-8.5.0/server/resources/log4j2.xml  \n  inflating: solr-8.5.0/server/scripts/cloud-scripts/zkcli.bat  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/antlr4-runtime-4.5.1-1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/asm-7.2.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/asm-commons-7.2.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/avatica-core-1.13.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/caffeine-2.8.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/calcite-core-1.18.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/calcite-linq4j-1.18.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/commons-cli-1.2.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/commons-codec-1.13.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/commons-collections-3.2.2.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/commons-compiler-3.0.9.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/commons-configuration2-2.1.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/commons-exec-1.3.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/commons-fileupload-1.3.3.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/commons-io-2.6.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/commons-lang3-3.9.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/commons-math3-3.6.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/commons-text-1.6.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/curator-client-2.13.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/curator-framework-2.13.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/curator-recipes-2.13.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/disruptor-3.4.2.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/eigenbase-properties-1.1.5.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/guava-25.1-jre.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/hadoop-annotations-3.2.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/hadoop-auth-3.2.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/hadoop-common-3.2.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/hadoop-hdfs-client-3.2.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/hppc-0.8.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/htrace-core4-4.1.0-incubating.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/http2-client-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/http2-common-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/http2-hpack-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/http2-http-client-transport-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/httpclient-4.5.10.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/httpcore-4.4.12.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/httpmime-4.5.10.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/jackson-annotations-2.10.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/jackson-core-2.10.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/jackson-databind-2.10.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/jackson-dataformat-smile-2.10.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/janino-3.0.9.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/java-semver-0.9.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/jetty-alpn-client-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/jetty-alpn-java-client-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/jetty-client-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/jetty-http-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/jetty-io-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/jetty-util-9.4.24.v20191120.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/jose4j-0.6.5.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/json-path-2.4.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/kerb-core-1.0.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/kerb-util-1.0.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/kerby-asn1-1.0.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/kerby-pkix-1.0.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-analyzers-common-8.5.0.jar  \n","name":"stdout"},{"output_type":"stream","text":"  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-analyzers-kuromoji-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-analyzers-nori-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-analyzers-phonetic-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-backward-codecs-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-classification-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-codecs-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-core-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-expressions-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-grouping-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-highlighter-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-join-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-memory-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-misc-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-queries-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-queryparser-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-sandbox-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-spatial-extras-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-spatial3d-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/lucene-suggest-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/netty-buffer-4.1.29.Final.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/netty-codec-4.1.29.Final.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/netty-common-4.1.29.Final.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/netty-handler-4.1.29.Final.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/netty-resolver-4.1.29.Final.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/netty-transport-4.1.29.Final.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/netty-transport-native-epoll-4.1.29.Final.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/netty-transport-native-unix-common-4.1.29.Final.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/opentracing-api-0.33.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/opentracing-noop-0.33.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/opentracing-util-0.33.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/org.restlet-2.4.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/org.restlet.ext.servlet-2.4.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/protobuf-java-3.11.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/re2j-1.2.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/rrd4j-3.5.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/s2-geometry-library-java-1.0.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/solr-core-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/solr-solrj-8.5.0.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/spatial4j-0.7.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/stax2-api-3.1.4.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/t-digest-3.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/woodstox-core-asl-4.4.1.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/zookeeper-3.5.5.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/lib/zookeeper-jute-3.5.5.jar  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/WEB-INF/web.xml  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/analysis.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/angular-csp.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/chosen.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/cloud.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/collections.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/common.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/cores.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/dashboard.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/dataimport.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/documents.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/files.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/index.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/java-properties.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/jquery-ui.min.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/jquery-ui.structure.min.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/jstree.style.min.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/logging.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/login.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/menu.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/overview.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/plugins.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/query.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/replication.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/schema.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/segments.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/stream.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/suggestions.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/css/angular/threads.css  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/favicon.ico  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/chosen-sprite.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/chosen-sprite@2x.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/div.gif  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/favicon.ico  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/7z.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/README  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/ai.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/aiff.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/asc.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/audio.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/bin.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/bz2.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/c.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/cfc.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/cfm.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/chm.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/class.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/conf.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/cpp.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/cs.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/css.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/csv.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/deb.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/divx.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/doc.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/dot.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/eml.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/enc.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/file.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/gif.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/gz.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/hlp.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/htm.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/html.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/image.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/iso.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/jar.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/java.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/jpeg.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/jpg.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/js.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/lua.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/m.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/mm.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/mov.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/mp3.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/mpg.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/odc.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/odf.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/odg.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/odi.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/odp.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/ods.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/odt.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/ogg.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/pdf.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/pgp.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/php.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/pl.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/png.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/ppt.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/ps.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/py.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/ram.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/rar.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/rb.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/rm.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/rpm.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/rtf.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/sig.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/sql.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/swf.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/sxc.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/sxd.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/sxi.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/sxw.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/tar.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/tex.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/tgz.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/txt.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/vcf.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/video.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/vsd.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/wav.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/wma.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/wmv.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/xls.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/xml.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/xpi.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/xvid.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/filetypes/zip.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/arrow-000-small.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/arrow-circle.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/arrow-switch.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/asterisk.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/battery.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/block-small.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/block.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/book-open-text.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/box.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/bug.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/chart.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/chevron-small-expand.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/chevron-small.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/clipboard-list.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/clipboard-paste-document-text.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/clipboard-paste.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/clock-select-remain.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/clock-select.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/construction.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/cross-0.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/cross-1.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/cross-button.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/cross.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/dashboard.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/database--plus.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/database.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/databases.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/disk-black.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/document-convert.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/document-import.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/document-list.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/document-text.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/documents-stack.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/download-cloud.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/drive-upload.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/exclamation-button.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/eye.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/folder-export.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/folder-tree.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/folder.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/funnel-small.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/funnel.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/gear.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/globe-network.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/globe.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/hammer-screwdriver.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/hammer.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/hand.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/highlighter-text.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/home.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/hourglass--exclamation.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/hourglass.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/idea.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/inbox-document-text.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/information-button.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/information-small.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/information-white.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/information.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/jar.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/magnifier.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/mail.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/memory.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/minus-button.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/molecule.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/network-cloud.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/network-status-away.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/network-status-busy.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/network-status-offline.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/network-status.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/network.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/node-design.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/node-master.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/node-select.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/node-slave.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/node.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/pencil-small.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/pencil.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/plus-button.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/processor.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/prohibition.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/property.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/question-small-white.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/question-white.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/question.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/receipt-invoice.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/receipt.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/run.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/script-code.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/server-cast.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/server.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/sitemap.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/slash.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/status-away.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/status-busy.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/status-offline.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/status.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/system-monitor--exclamation.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/system-monitor.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/table.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/terminal.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/tick-circle.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/tick-red.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/tick.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/toggle-small-expand.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/toggle-small.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/toolbox.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/ui-accordion.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/ui-address-bar.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/ui-check-box-uncheck.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/ui-check-box.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/ui-radio-button-uncheck.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/ui-radio-button.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/ui-text-field-select.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/users.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/wooden-box.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/ico/zone.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/jstree/32px.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/jstree/40px.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/jstree/throbber.gif  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/loader-light.gif  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/loader.gif  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/lucene-ico.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/solr-ico.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/solr.svg  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/img/tree.png  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/index.html  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/app.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/alias-overview.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/analysis.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/cloud.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/cluster-suggestions.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/collection-overview.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/collections.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/core-overview.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/cores.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/dataimport.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/documents.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/files.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/index.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/java-properties.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/logging.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/login.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/plugins.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/query.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/replication.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/schema.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/segments.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/stream.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/threads.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/controllers/unknown.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/js/angular/services.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/angular-chosen.min.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/angular-cookies.min.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/angular-resource.min.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/angular-route.min.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/angular-sanitize.min.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/angular-utf8-base64.min.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/angular.min.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/chosen.jquery.min.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/d3.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/highlight.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/jquery-3.4.1.min.js  \n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/jquery-ui.min.js  \n","name":"stdout"},{"output_type":"stream","text":"  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/jstree.min.js  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/libs/ngtimeago.js  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/alias_overview.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/analysis.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/cloud.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/cluster_suggestions.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/collection_overview.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/collections.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/core_overview.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/cores.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/dataimport.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/documents.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/files.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/index.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/java-properties.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/logging-levels.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/logging.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/login.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/plugins.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/query.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/replication.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/schema.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/segments.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/stream.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/threads.html  \r\n  inflating: solr-8.5.0/server/solr-webapp/webapp/partials/unknown.html  \r\n  inflating: solr-8.5.0/server/solr/README.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/contractions_ca.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/contractions_fr.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/contractions_ga.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/contractions_it.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/hyphenations_ga.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stemdict_nl.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stoptags_ja.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_ar.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_bg.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_ca.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_cz.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_da.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_de.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_el.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_en.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_es.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_et.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_eu.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_fa.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_fi.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_fr.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_ga.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_gl.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_hi.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_hu.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_hy.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_id.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_it.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_ja.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_lv.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_nl.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_no.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_pt.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_ro.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_ru.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_sv.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_th.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/stopwords_tr.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/lang/userdict_ja.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/managed-schema  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/protwords.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/solrconfig.xml  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/stopwords.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/_default/conf/synonyms.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/_rest_managed.json  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/_schema_analysis_stopwords_english.json  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/_schema_analysis_synonyms_english.json  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/clustering/carrot2/README.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/clustering/carrot2/kmeans-attributes.xml  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/clustering/carrot2/lingo-attributes.xml  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/clustering/carrot2/stc-attributes.xml  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/currency.xml  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/elevate.xml  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/contractions_ca.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/contractions_fr.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/contractions_ga.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/contractions_it.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/hyphenations_ga.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stemdict_nl.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stoptags_ja.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_ar.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_bg.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_ca.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_ckb.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_cz.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_da.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_de.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_el.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_en.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_es.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_et.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_eu.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_fa.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_fi.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_fr.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_ga.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_gl.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_hi.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_hu.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_hy.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_id.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_it.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_ja.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_lv.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_nl.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_no.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_pt.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_ro.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_ru.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_sv.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_th.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/stopwords_tr.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/lang/userdict_ja.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/managed-schema  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/mapping-FoldToASCII.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/mapping-ISOLatin1Accent.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/params.json  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/protwords.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/solrconfig.xml  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/spellings.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/stopwords.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/synonyms.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/update-script.js  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/README.txt  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/VM_global_library.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/browse.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/cluster.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/cluster_results.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/debug.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/did_you_mean.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/error.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/facet_fields.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/facet_pivot.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/facet_queries.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/facet_ranges.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/facets.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/footer.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/head.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/header.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/hit.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/hit_grouped.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/hit_plain.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/join_doc.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/jquery.autocomplete.css  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/jquery.autocomplete.js  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/layout.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/main.css  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/mime_type_lists.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/pagination_bottom.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/pagination_top.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/product_doc.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/query.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/query_form.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/query_group.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/query_spatial.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/results_list.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/richtext_doc.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/suggest.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/velocity/tabs.vm  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/xslt/example.xsl  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/xslt/example_atom.xsl  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/xslt/example_rss.xsl  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/xslt/luke.xsl  \r\n  inflating: solr-8.5.0/server/solr/configsets/sample_techproducts_configs/conf/xslt/updateXml.xsl  \r\n  inflating: solr-8.5.0/server/solr/solr.xml  \r\n  inflating: solr-8.5.0/server/solr/zoo.cfg  \r\n  inflating: solr-8.5.0/server/start.jar  \r\n   creating: solr-8.5.0/bin/\r\n   creating: solr-8.5.0/bin/init.d/\r\n   creating: solr-8.5.0/contrib/prometheus-exporter/bin/\r\n  inflating: solr-8.5.0/bin/init.d/solr  \r\n  inflating: solr-8.5.0/bin/install_solr_service.sh  \r\n  inflating: solr-8.5.0/bin/oom_solr.sh  \r\n  inflating: solr-8.5.0/bin/post     \r\n  inflating: solr-8.5.0/bin/postlogs  \r\n  inflating: solr-8.5.0/bin/solr     \r\n  inflating: solr-8.5.0/bin/solr.cmd  \r\n  inflating: solr-8.5.0/bin/solr.in.cmd  \r\n  inflating: solr-8.5.0/bin/solr.in.sh  \r\n  inflating: solr-8.5.0/contrib/prometheus-exporter/bin/solr-exporter  \r\n  inflating: solr-8.5.0/contrib/prometheus-exporter/bin/solr-exporter.cmd  \r\n  inflating: solr-8.5.0/example/exampledocs/test_utf8.sh  \r\n  inflating: solr-8.5.0/server/scripts/cloud-scripts/snapshotscli.sh  \r\n  inflating: solr-8.5.0/server/scripts/cloud-scripts/zkcli.sh  \r\n","name":"stdout"}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"!ls","execution_count":4,"outputs":[{"output_type":"stream","text":"__notebook_source__.ipynb  solr-8.5.0  solr-8.5.0.zip\r\n","name":"stdout"}]},{"metadata":{},"cell_type":"markdown","source":"### Data Preprocessing"},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"# This Python 3 environment comes with many helpful analytics libraries installed\n# It is defined by the kaggle/python docker image: https://github.com/kaggle/docker-python\n# For example, here's several helpful packages to load in \n\n# import numpy as np # linear algebra\n# import pandas as pd # data processing, CSV file I/O (e.g. pd.read_csv)\n\n# Input data files are available in the \"../input/\" directory.\n# For example, running this (by clicking run or pressing Shift+Enter) will list all files under the input directory\n\n# import os\n# for dirname, _, filenames in os.walk('/kaggle/input'):\n#     for filename in filenames:\n#         print(os.path.join(dirname, filename))\n\n# Any results you write to the current directory are saved as output.\nimport os\nimport json\nimport glob\nimport re\nimport time\nimport pysolr\nimport csv\nimport time\nimport scipy\nimport spacy\nimport scispacy\nimport pandas as pd\nimport numpy as np\nimport networkx as nx\nimport ipywidgets as widgets\nfrom os import path\nfrom pandas import ExcelWriter\nfrom pandas import ExcelFile\nfrom jsonpath_ng.ext import parse\nfrom collections import Counter\nfrom collections import OrderedDict\nfrom IPython.core.display import display, HTML\nfrom pyvis.network import Network\nfrom datetime import date\nimport dateutil.parser as dparser\nimport seaborn as sns\nimport matplotlib.pyplot as plt\n\n%matplotlib inline\n\nimport warnings\nwarnings.filterwarnings('ignore')","execution_count":5,"outputs":[]},{"metadata":{},"cell_type":"markdown","source":"### Preprocessing data"},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"def clean_text(text) :\n    text = re.sub(r\" ?\\([^)]*\\)\", \"\", text)\n    text = re.sub(r\" ?\\[[^)]*\\]\", \"\", text)\n    text = re.sub(r\"http\\S+\", \"\", text)\n    text = re.sub(r\"  \", \" \", text)\n    return text","execution_count":6,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"def preprocess_data() :\n    start = time.time()\n\n    meta_df = pd.read_csv('/kaggle/input/CORD-19-research-challenge/metadata.csv', sep=',', header=0)\n    meta_columns = list(meta_df.columns)\n    meta_df.fillna('', inplace=True)\n\n    # Filter to pick only needed sections\n    include_set = ['Abstract','Introduction', 'background', 'Discussion', 'Results', 'Results and Discussion', 'methods,results']\n    section_df = pd.read_excel('/kaggle/input/covidsectionheads/covid_section_heads.xlsx', sheet_name='Sheet1')\n    section_df.dropna(subset=['Section Heads'], inplace=True)\n    section_df = section_df[section_df['Section Heads'].isin(include_set)]\n\n    # Categories/Targets to pick from Section Heads column\n    introduction_categories = [\"Introduction\"]\n    discussion_categories = [\"Discussion\"]\n    result_categories = [\"Results\", \"Results and Discussion\", 'methods,results']\n\n    introduction_df = section_df[section_df['Section Heads'].isin(introduction_categories)]\n    discussion_df = section_df[section_df['Section Heads'].isin(discussion_categories)]\n    result_df = section_df[section_df['Section Heads'].isin(result_categories)]\n\n    intro_list = introduction_df.iloc[:, 0].tolist()\n    discussion_list = discussion_df.iloc[:, 0].tolist()\n    result_list = result_df.iloc[:, 0].tolist()\n\n    intro_list = list(map(lambda x: str(x).strip(), intro_list))\n    discussion_list = list(map(lambda x: str(x).strip(), discussion_list))\n    result_list = list(map(lambda x: str(x).strip(), result_list))\n\n    path = '/kaggle/input/CORD-19-research-challenge/'\n    paths = [p for p in glob.glob(path + \"**/*.json\", recursive=True)]\n    files_size = len(paths)\n\n    col_names = ['paper_id','title','source', 'abstract','introduction','discussion','result','body', 'has_covid']\n    clean_df = pd.DataFrame(columns=col_names)\n\n    covid_syns = ['COVID-19','COVID19','2019-nCoV','2019nCoV','Coronavirus','SARS-CoV-2','SARSCov2','novel Coronavirus']\n\n    target_empty_count = 0\n\n    abstract_expr = parse('$.abstract[*].text')\n\n    for path in paths:\n        with open(path) as f:\n            intro_text_list = list()\n            discussion_text_list = list()\n            result_text_list = list()\n\n            data = json.load(f)\n\n            abstract_texts = [match.value for match in abstract_expr.find(data)]\n\n            body_nodes = data['body_text']\n\n            for entry in body_nodes :\n                section_name = entry['section']\n                section_name = section_name.strip().lower()\n                entry_text = entry['text']\n\n                if section_name.strip() in intro_list:\n                    intro_text_list.append(entry_text)\n\n                if section_name.strip() in discussion_list:\n                    discussion_text_list.append(entry_text)\n\n                if section_name.strip() in result_list:\n                    result_text_list.append(entry_text)\n\n            if len(intro_text_list) == 0 and len(discussion_text_list) == 0 and len(result_text_list) == 0 :\n                target_empty_count = target_empty_count + 1\n\n\n            id = data['paper_id']\n            title = data['metadata']['title']\n\n            pubtime_df = meta_df[meta_df.sha == id]['publish_time']\n            pubtime_dict = pubtime_df.to_dict()\n            pubtime = ''\n            for pubtime_field_key in pubtime_dict.keys():\n                temp_pubtime_str = pubtime_dict.get(pubtime_field_key)\n                orig_temp_pubtime_str = temp_pubtime_str\n                try:\n                    temppubdate = dparser.parse(orig_temp_pubtime_str,fuzzy=True).date()\n                    pubtime = temppubdate.strftime(\"%Y-%m-%dT%H:%M:%SZ\")\n                except Exception as e:\n                    temp_pubtime_str_parts = temp_pubtime_str.split(' ')\n                    if len(temp_pubtime_str_parts) > 2 :\n                        try :\n                            temp_pubtime_str = temp_pubtime_str_parts[0] + ' ' + temp_pubtime_str_parts[1] + ' ' + temp_pubtime_str_parts[2]\n                            temppubdate = dparser.parse(temp_pubtime_str,fuzzy=True).date()\n                            pubtime = temppubdate.strftime(\"%Y-%m-%dT%H:%M:%SZ\")\n                        except Exception as ex:\n                            pubtime = ''\n                    else:\n                        pubtime = ''      \n\n            sha_df = meta_df[meta_df.sha == id]['source_x']\n            meta_dict = sha_df.to_dict()\n            source = ''\n            for meta_field_key in meta_dict.keys():\n                source = meta_dict.get(meta_field_key)\n\n            if not source:\n                title_df = meta_df[meta_df.title == title]['source_x']\n                meta_dict = title_df.to_dict()\n                for meta_field_key in meta_dict.keys():\n                    source = meta_dict.get(meta_field_key)\n\n            abstract = clean_text(\" \".join(abstract_texts))\n            introduction = clean_text(\" \".join(intro_text_list))\n            discussion = clean_text(\" \".join(discussion_text_list))\n            result = clean_text(\" \".join(result_text_list))\n            body = \" \".join([introduction, discussion, result])\n\n            has_covid = 'false'\n\n            res = [ele for ele in covid_syns if (ele.lower() in body.lower())]\n            if(len(res)  > 0):\n                has_covid = 'true'\n\n            if len(body.strip()) > 0 or len(abstract) > 0:\n                new_row = {'paper_id': id, 'title': title.strip(), 'source': source,'abstract': abstract.strip(),\n                           'introduction': introduction.strip(),'discussion': discussion.strip(),\n                           'result': result.strip(), 'body': body.strip(), 'publish_time': pubtime,'has_covid': has_covid}\n                clean_df = clean_df.append(new_row, ignore_index=True)\n\n    # Drop duoplicate papers\n    clean_df.drop_duplicates(subset=['title','abstract'], keep='first', inplace=False)\n    clean_df.to_csv('/kaggle/working/CORD-19.csv', index=True)\n\n    print('Final DataFrame Shape - ', clean_df.shape)\n    print(\"Papers that dont have Intro, Discussion or Result  - \", target_empty_count)\n    print('Total Papers processed - ', files_size)\n\n    print('Time Elaspsed - ', time.time() - start)\n    ","execution_count":7,"outputs":[]},{"metadata":{},"cell_type":"markdown","source":"### Configure Search Engine"},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"!solr-8.5.0/bin/solr start -force","execution_count":8,"outputs":[{"output_type":"stream","text":"NOTE: Please install lsof as this script needs it to determine if Solr is listening on port 8983.\n\nStarted Solr server on port 8983 (pid=222). Happy searching!\n\n","name":"stdout"}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"!solr-8.5.0/bin/solr create -c covid19 -s 1 -rf 1 -force","execution_count":9,"outputs":[{"output_type":"stream","text":"WARNING: Using _default configset with data driven schema functionality. NOT RECOMMENDED for production use.\n         To turn off: bin/solr config -c covid19 -p 8983 -action set-user-property -property update.autoCreateFields -value false\n\nCreated new core 'covid19'\n","name":"stdout"}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"# Using _default configset with data driven schema functionality. NOT RECOMMENDED for production use.\n!solr-8.5.0/bin/solr config -c covid19 -p 8983 -action set-user-property -property update.autoCreateFields -value false","execution_count":10,"outputs":[{"output_type":"stream","text":"\nPOSTing request to Config API: http://localhost:8983/solr/covid19/config\n{\"set-user-property\":{\"update.autoCreateFields\":\"false\"}}\nSuccessfully set-user-property update.autoCreateFields to false\n","name":"stdout"}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"#Set Up Synonyms\n\n!echo 'COVID-19,covid19,2019-nCoV,2019nCoV,Coronavirus,SARS-CoV-2,SARSCov2,novel Coronavirus' >> solr-8.5.0/server/solr/covid19/conf/synonyms.txt;\n!echo 'heart,cardiac,tachycardia,myocardial' >> solr-8.5.0/server/solr/covid19/conf/synonyms.txt;\n!echo 'pulmonary,respiratory' >> solr-8.5.0/server/solr/covid19/conf/synonyms.txt;","execution_count":11,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"!cat solr-8.5.0/server/solr/covid19/conf/synonyms.txt","execution_count":12,"outputs":[{"output_type":"stream","text":"# The ASF licenses this file to You under the Apache License, Version 2.0\r\r\n# (the \"License\"); you may not use this file except in compliance with\r\r\n# the License.  You may obtain a copy of the License at\r\r\n#\r\r\n#     http://www.apache.org/licenses/LICENSE-2.0\r\r\n#\r\r\n# Unless required by applicable law or agreed to in writing, software\r\r\n# distributed under the License is distributed on an \"AS IS\" BASIS,\r\r\n# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\r\r\n# See the License for the specific language governing permissions and\r\r\n# limitations under the License.\r\r\n\r\r\n#-----------------------------------------------------------------------\r\r\n#some test synonym mappings unlikely to appear in real input text\r\r\naaafoo => aaabar\r\r\nbbbfoo => bbbfoo bbbbar\r\r\ncccfoo => cccbar cccbaz\r\r\nfooaaa,baraaa,bazaaa\r\r\n\r\r\n# Some synonym groups specific to this example\r\r\nGB,gib,gigabyte,gigabytes\r\r\nMB,mib,megabyte,megabytes\r\r\nTelevision, Televisions, TV, TVs\r\r\n#notice we use \"gib\" instead of \"GiB\" so any WordDelimiterGraphFilter coming\r\r\n#after us won't split it into two words.\r\r\n\r\r\n# Synonym mappings can be used for spelling correction too\r\r\npixima => pixma\r\r\n\r\r\nCOVID-19,covid19,2019-nCoV,2019nCoV,Coronavirus,SARS-CoV-2,SARSCov2,novel Coronavirus\r\nheart,cardiac,tachycardia,myocardial\r\npulmonary,respiratory\r\n","name":"stdout"}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"#Reload the covid19 core/collection because we added new synonyms. Need reload as it will affect index\n#Whenever new synonyms are added we need to reindex as synonyms are applied both on index and query analyzers\n!curl 'http://localhost:8983/solr/admin/cores?action=RELOAD&core=covid19'","execution_count":13,"outputs":[{"output_type":"stream","text":"{\r\n  \"responseHeader\":{\r\n    \"status\":0,\r\n    \"QTime\":433}}\r\n","name":"stdout"}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"#Add custom field Type that wont tokenize phrases for fields like source etc\n!curl -X POST -H 'Content-type:application/json' --data-binary '{\"add-field-type\" : {\"name\":\"keywordText\",\"class\":\"solr.TextField\", \"positionIncrementGap\":\"100\", \"indexAnalyzer\" : {\"tokenizer\":{\"class\":\"solr.KeywordTokenizerFactory\" }, \"filters\":[{\"class\":\"solr.TrimFilterFactory\"},{\"class\":\"solr.StopFilterFactory\", \"ignoreCase\":true, \"words\":\"lang/stopwords_en.txt\"},{\"class\":\"solr.ManagedSynonymGraphFilterFactory\", \"managed\":\"english\" },{\"class\":\"solr.RemoveDuplicatesTokenFilterFactory\"},{\"class\":\"solr.FlattenGraphFilterFactory\"}]},\"queryAnalyzer\" : {\"tokenizer\":{\"class\":\"solr.KeywordTokenizerFactory\" },\"filters\":[{\"class\":\"solr.TrimFilterFactory\"},{\"class\":\"solr.StopFilterFactory\", \"ignoreCase\":true, \"words\":\"lang/stopwords_en.txt\"},{\"class\":\"solr.ManagedSynonymGraphFilterFactory\", \"managed\":\"english\" },{\"class\":\"solr.RemoveDuplicatesTokenFilterFactory\"}]}}}' http://localhost:8983/solr/covid19/schema","execution_count":14,"outputs":[{"output_type":"stream","text":"{\r\n  \"responseHeader\":{\r\n    \"status\":0,\r\n    \"QTime\":527}}\r\n","name":"stdout"}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"#Create SOLR field definitions\n!curl -X POST -H 'Content-type:application/json' --data-binary '{\"add-field\": {\"name\":\"title\", \"type\":\"text_en_splitting_tight\", \"multiValued\":false, \"stored\":true, \"indexed\":true}}' http://localhost:8983/solr/covid19/schema;\n!curl -X POST -H 'Content-type:application/json' --data-binary '{\"add-field\": {\"name\":\"abstract\", \"type\":\"text_en_splitting_tight\", \"multiValued\":false, \"stored\":true, \"indexed\":true}}' http://localhost:8983/solr/covid19/schema;\n!curl -X POST -H 'Content-type:application/json' --data-binary '{\"add-field\": {\"name\":\"source\", \"type\":\"keywordText\", \"multiValued\":false, \"stored\":true, \"indexed\":true}}' http://localhost:8983/solr/covid19/schema;\n!curl -X POST -H 'Content-type:application/json' --data-binary '{\"add-field\": {\"name\":\"introduction\", \"type\":\"text_en_splitting_tight\", \"multiValued\":false, \"stored\":true, \"indexed\":true}}' http://localhost:8983/solr/covid19/schema;\n!curl -X POST -H 'Content-type:application/json' --data-binary '{\"add-field\": {\"name\":\"discussion\", \"type\":\"text_en_splitting_tight\", \"multiValued\":false, \"stored\":true, \"indexed\":true}}' http://localhost:8983/solr/covid19/schema;\n!curl -X POST -H 'Content-type:application/json' --data-binary '{\"add-field\": {\"name\":\"result\", \"type\":\"text_en_splitting_tight\", \"multiValued\":false, \"stored\":true, \"indexed\":true}}' http://localhost:8983/solr/covid19/schema;\n!curl -X POST -H 'Content-type:application/json' --data-binary '{\"add-field\": {\"name\":\"body\", \"type\":\"text_en_splitting_tight\", \"multiValued\":false, \"stored\":true, \"indexed\":true}}' http://localhost:8983/solr/covid19/schema;\n!curl -X POST -H 'Content-type:application/json' --data-binary '{\"add-field\": {\"name\":\"publish_time\", \"type\":\"pdate\", \"multiValued\":false, \"stored\":true, \"indexed\":true}}' http://localhost:8983/solr/covid19/schema;\n!curl -X POST -H 'Content-type:application/json' --data-binary '{\"add-field\": {\"name\":\"has_covid\", \"type\":\"boolean\", \"multiValued\":false, \"stored\":true, \"indexed\":true}}' http://localhost:8983/solr/covid19/schema;","execution_count":15,"outputs":[{"output_type":"stream","text":"{\n  \"responseHeader\":{\n    \"status\":0,\n    \"QTime\":349}}\n{\n  \"responseHeader\":{\n    \"status\":0,\n    \"QTime\":375}}\n{\n  \"responseHeader\":{\n    \"status\":0,\n    \"QTime\":400}}\n{\n  \"responseHeader\":{\n    \"status\":0,\n    \"QTime\":373}}\n{\n  \"responseHeader\":{\n    \"status\":0,\n    \"QTime\":329}}\n{\n  \"responseHeader\":{\n    \"status\":0,\n    \"QTime\":292}}\n{\n  \"responseHeader\":{\n    \"status\":0,\n    \"QTime\":310}}\n{\n  \"responseHeader\":{\n    \"status\":0,\n    \"QTime\":308}}\n{\n  \"responseHeader\":{\n    \"status\":0,\n    \"QTime\":308}}\n","name":"stdout"}]},{"metadata":{},"cell_type":"markdown","source":"### Create Search Index"},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"solr = pysolr.Solr('http://localhost:8983/solr/covid19/', timeout=10)","execution_count":16,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"generic_model = spacy.load('en_core_sci_md')\n\n#Load preprocessed CSV data\ncsv_path = '/kaggle/working/CORD-19.csv'\n\nif not path.exists(csv_path):\n    preprocess_data()\n    \ndf = pd.read_csv(csv_path, sep=',', header=0)\ndf.fillna('', inplace=True)\nprint('DF candidate_list size - ', df.shape)\n\ndf.head(2)","execution_count":19,"outputs":[{"output_type":"stream","text":"DF candidate_list size -  (43464, 11)\n","name":"stdout"},{"output_type":"execute_result","execution_count":19,"data":{"text/plain":"   Unnamed: 0                                  paper_id  \\\n0           0  d99acb4e99be7852aa61a688c9fbd38d44b5a252   \n1           1  748d4c57fe1acc8d9d97cf574f7dea5296f9386c   \n\n                                               title source  \\\n0  Evaluation of Measles Vaccine Virus as a Vecto...    PMC   \n1  Direct visualization of Ebola virus fusion tri...    PMC   \n\n                                            abstract  \\\n0  Live attenuated recombinant measles vaccine vi...   \n1  Ebola virus makes extensive and intricate use ...   \n\n                                        introduction  \\\n0  Live attenuated viruses have been developed an...   \n1                                                      \n\n                                          discussion result  \\\n0  Reverse genetics has been developed for genera...          \n1  Most previous attempts to characterize EBOV me...          \n\n                                                body  has_covid  \\\n0  Live attenuated viruses have been developed an...      False   \n1  Most previous attempts to characterize EBOV me...      False   \n\n           publish_time  \n0  2012-02-16T00:00:00Z  \n1  2016-02-09T00:00:00Z  ","text/html":"<div>\n<style scoped>\n    .dataframe tbody tr th:only-of-type {\n        vertical-align: middle;\n    }\n\n    .dataframe tbody tr th {\n        vertical-align: top;\n    }\n\n    .dataframe thead th {\n        text-align: right;\n    }\n</style>\n<table border=\"1\" class=\"dataframe\">\n  <thead>\n    <tr style=\"text-align: right;\">\n      <th></th>\n      <th>Unnamed: 0</th>\n      <th>paper_id</th>\n      <th>title</th>\n      <th>source</th>\n      <th>abstract</th>\n      <th>introduction</th>\n      <th>discussion</th>\n      <th>result</th>\n      <th>body</th>\n      <th>has_covid</th>\n      <th>publish_time</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>0</th>\n      <td>0</td>\n      <td>d99acb4e99be7852aa61a688c9fbd38d44b5a252</td>\n      <td>Evaluation of Measles Vaccine Virus as a Vecto...</td>\n      <td>PMC</td>\n      <td>Live attenuated recombinant measles vaccine vi...</td>\n      <td>Live attenuated viruses have been developed an...</td>\n      <td>Reverse genetics has been developed for genera...</td>\n      <td></td>\n      <td>Live attenuated viruses have been developed an...</td>\n      <td>False</td>\n      <td>2012-02-16T00:00:00Z</td>\n    </tr>\n    <tr>\n      <th>1</th>\n      <td>1</td>\n      <td>748d4c57fe1acc8d9d97cf574f7dea5296f9386c</td>\n      <td>Direct visualization of Ebola virus fusion tri...</td>\n      <td>PMC</td>\n      <td>Ebola virus makes extensive and intricate use ...</td>\n      <td></td>\n      <td>Most previous attempts to characterize EBOV me...</td>\n      <td></td>\n      <td>Most previous attempts to characterize EBOV me...</td>\n      <td>False</td>\n      <td>2016-02-09T00:00:00Z</td>\n    </tr>\n  </tbody>\n</table>\n</div>"},"metadata":{}}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"# Index each pandas row as a document into SOLR search engine\n\ncovid_syns = ('SARSCoV2','SARS-CoV-2', '2019-nCoV','2019nCoV','COVID-19', 'COVID19','coronavirus', 'corona virus' 'novel coronavirus')\n\nlist_for_solr=[]\ncounter = 0\nfor index, row in df.iterrows():\n    id = row['paper_id']\n    title = row[\"title\"]\n    source = row[\"source\"]    \n    abstract = row[\"abstract\"]\n    introduction = row[\"introduction\"]\n    discussion = row[\"discussion\"]\n    result = row[\"result\"]\n    publish_time = row[\"publish_time\"]    \n    body = row[\"body\"]  # Cocatenated text of all text fields abstract, introduction, discussion, result\n    \n    if((title and title.isspace()) and (abstract and abstract.isspace()) and (body and body.isspace())):\n        continue\n        \n    has_covid = 'false'\n    if any(words in body for words in covid_syns):\n        has_covid = 'true'\n    \n    solr_content = {}\n    solr_content['id'] = id\n    solr_content['title'] = title\n    solr_content['source'] = source\n    solr_content['abstract'] = abstract\n    solr_content['introduction'] = introduction\n    solr_content['discussion'] = discussion\n    solr_content['result'] = result    \n    solr_content['body'] = body\n    solr_content['has_covid'] = has_covid\n    solr_content['publish_time'] = publish_time    \n        \n    list_for_solr.append(solr_content)\n    \n    if index % 1000 == 0:\n        solr.add(list_for_solr)\n        list_for_solr = []\n        counter = counter + 1000\n        print('Counter ', counter)\n        \n#Commit is very costly use it sparingly        \nsolr.commit()\nprint('Indexing Finished !')","execution_count":20,"outputs":[{"output_type":"stream","text":"Counter  1000\nCounter  2000\nCounter  3000\nCounter  4000\nCounter  5000\nCounter  6000\nCounter  7000\nCounter  8000\nCounter  9000\nCounter  10000\nCounter  11000\nCounter  12000\nCounter  13000\nCounter  14000\nCounter  15000\nCounter  16000\nCounter  17000\nCounter  18000\nCounter  19000\nCounter  20000\nCounter  21000\nCounter  22000\nCounter  23000\nCounter  24000\nCounter  25000\nCounter  26000\nCounter  27000\nCounter  28000\nCounter  29000\nCounter  30000\nCounter  31000\nCounter  32000\nCounter  33000\nCounter  34000\nCounter  35000\nCounter  36000\nCounter  37000\nCounter  38000\nCounter  39000\nCounter  40000\nCounter  41000\nCounter  42000\nCounter  43000\nCounter  44000\nIndexing Finished !\n","name":"stdout"}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"def extract_entities(models, text) :\n    entities = {}\n    \n    for nlp in models :\n        doc = nlp(text)\n        for ent in doc.ents:\n            entity = ent.text\n            if ent.label_ in entities :\n                if entities[ent.label_].count(ent.text) == 0:\n                    entities[ent.label_].append(ent.text)\n            else :\n                entities[ent.label_] = [ent.text]\n\n    return entities","execution_count":21,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"def initilize_nlp_models(model_names):\n    models = {}\n    for name in model_names:\n        models[name] = spacy.load(name)\n    \n    print('Models Loaded')\n    return models","execution_count":22,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"def search_task_answers(search_results) :\n    answers_list = list()\n    \n    for search_result in search_results:\n        doc_hl_dict = {}\n        \n        id = search_result.get('id', \"MISSING\")\n        title = search_result.get('title', \"MISSING\")\n        \n        doc_highlights = search_results.highlighting[id]\n        \n        doc_hl_dict['id'] = id\n        doc_hl_dict['title'] = title\n        \n        if len(doc_highlights) > 0:\n            display(HTML(f'<h4><i>{title}\\n</i></h4>'))\n\n        for doc_hl_field in doc_highlights:\n            hl_snippets = doc_highlights[doc_hl_field]\n        \n            if len(hl_snippets) > 0 :\n                answer_snippet = ''\n                \n                #print('ID : ', id, '\\nTITLE : ', title)\n                #print('\\t', doc_hl_field)\n                display(HTML(f'\\t<h5>{doc_hl_field}\\n</h5>\\n'))\n            \n                for index, snippet in enumerate(hl_snippets, start=1):\n                    answer_snippet = answer_snippet.strip() + \" \" + snippet.strip()\n                    \n                    display(HTML(f'<blockquote>{index}. {snippet.strip()}\\n</blockquote>'))\n                    #print('\\t\\t', index , '. ' , snippet.strip(), '\\n') \n                                  \n                doc_hl_dict[doc_hl_field] = answer_snippet.strip()\n                    \n        if len(doc_hl_dict) > 0:\n            answers_list.append(doc_hl_dict)\n        \n    return answers_list","execution_count":23,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"def search(query, rows=5):\n    # Search for data\n    search_results = solr.search(query, **{\n    'rows':rows,\n    'fq':'has_covid:true',\n    'qf':'title^50.0 abstract^40.0 introduction^30.0 discussion^20.0 result^50.0 body^10.0',\n    'pf':'title^60.0 abstract^50.0 introduction^40.0 discussion^30.0 result^60.0 body^20.0',\n    'hl': 'true',\n    'hl.bs.type': 'SENTENCE',\n    'hl.method' : 'unified',\n    'hl.snippets' : 5,\n    'hl.usePhraseHighlighter': 'true',\n    'hl.highlightMultiTerm' : 'true',\n    'hl.tag.pre':'',\n    'hl.tag.post':'',\n    'df':'body',\n    'hl.fl':'introduction,discussion,result'\n    })\n\n    num_docs_found = search_results.hits\n    num_search_results = len(search_results)\n    display(HTML(f'<h3 style=\"color:blue\">Top {num_search_results} search result(s) of {num_docs_found} total. \\n</h3>'))\n    \n    return num_docs_found, search_results","execution_count":36,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"def populate_labels(task_answers) :\n    field_list = ['introduction','discussion', 'result']\n\n    for doc_answer_dict in task_answers:\n        all_entities = set()\n        intro_entities = set()\n        discussion_entities = set()\n        result_entities = set() \n\n        for field_name, answer_text in doc_answer_dict.items():\n            if field_name in field_list:\n                ent_dict = extract_entities(chosen_models, answer_text)\n                ent_list = ent_dict['ENTITY']\n\n                all_entities.update(ent_list)\n                if field_name == 'introduction' :\n                    intro_entities.update(ent_list)\n                elif field_name == 'discussion' :\n                    discussion_entities.update(ent_list)\n                else :\n                    result_entities.update(ent_list)\n\n        # Now set up labels for entities\n        prior_newdata_entities = set()\n        prior_strong_entities = set()\n        prior_entities = set()\n        speculative_entities = set()\n        unknown_entities = set()    \n        novel_entities = set()\n\n        for a_ent in all_entities :\n            if a_ent in intro_entities and a_ent in result_entities and a_ent in discussion_entities : \n                prior_newdata_entities.add(a_ent)\n            elif a_ent in intro_entities and a_ent not in result_entities and a_ent in discussion_entities :\n                prior_strong_entities.add(a_ent)\n            elif a_ent in intro_entities and a_ent not in result_entities and a_ent not in discussion_entities :\n                prior_entities.add(a_ent)\n            elif a_ent not in intro_entities and a_ent in result_entities and a_ent not in discussion_entities : \n                unknown_entities.add(a_ent)\n            elif a_ent not in intro_entities and a_ent in result_entities and a_ent in discussion_entities :\n                novel_entities.add(a_ent)\n            else :\n                pass\n            \n        if(len(prior_newdata_entities) > 0) :\n            doc_answer_dict['prior-newdata'] = list(prior_newdata_entities)\n\n        if(len(prior_strong_entities) > 0) :\n            doc_answer_dict['prior-strong'] = list(prior_strong_entities)          \n\n        if(len(prior_entities) > 0) :\n            doc_answer_dict['prior'] = list(prior_entities)             \n\n        if(len(speculative_entities) > 0) :\n            doc_answer_dict['speculative'] = list(speculative_entities)\n\n        if(len(unknown_entities) > 0) :\n            doc_answer_dict['unknown'] = list(unknown_entities)           \n\n        if(len(novel_entities) > 0) :\n            doc_answer_dict['novel'] = list(novel_entities)           \n\n    return task_answers","execution_count":37,"outputs":[]},{"metadata":{},"cell_type":"markdown","source":"### What do we know about COVID-19 risk factors?\n\nTask Details\n\nWhat do we know about COVID-19 risk factors? What have we learned from epidemiological studies?\nSpecifically, we want to know what the literature reports about:\n\n* Data on potential risks factors\n    1. Smoking, pre-existing pulmonary disease\n    2. Co-infections (determine whether co-existing respiratory/viral infections make the virus more transmissible or virulent) and other co-morbidities\n    3. Neonates and pregnant women\n    4. Socio-economic and behavioral factors to understand the economic impact of the virus and whether there were differences.\n* Transmission dynamics of the virus, including the basic reproductive number, incubation period, serial interval, modes of transmission and environmental factors\n* Severity of disease, including risk of fatality among symptomatic hospitalized patients, and high-risk patient groups\n* Susceptibility of populations\n* Public health mitigation measures that could be effective for control\n"},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"tasks = ['Smoking and pre-existing pulmonary disease', \n         'Co-infections, co-morbidities and respiratory infections',\n         'Neonates and pregnant women',\n         'Socio-economic and behavioral factors to understand the economic impact of the virus and whether there were differences.',\n         'Transmission dynamics of the virus, including the basic reproductive number, incubation period, serial interval, modes of transmission and environmental factors',\n         'Severity of disease, including risk of fatality among symptomatic hospitalized patients, and high-risk patient groups',\n         'Susceptibility of populations',\n         'Public health mitigation measures that could be effective for control'\n        ]\n\nqueries = ['(\"Smoking COVID-19\"~10 OR \"pulmonary disease\"~10)', \n           '(\"Co-infections COVID-19\"~10 OR \"co-morbidities COVID-19\"~10 OR \"respiratory infections COVID-19\"~10)',\n           '(\"Neonates COVID-19\"~10 OR \"pregnant women COVID-19\"~10)',\n           'Socio-economic, behavioral factors and economic impact',\n           'Transmission dynamics, basic reproductive number, incubation period, serial interval, modes of transmission and environmental factors',\n           'Severity of disease, risk of fatality among symptomatic hospitalized patients and high-risk patient groups',\n           'Susceptibility of populations',\n           'Public health mitigation measures that are effective for control'\n        ]","execution_count":32,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"#Load NLP Models only once\n# model_names = ['en_ner_craft_md','en_ner_jnlpba_md', 'en_ner_bc5cdr_md','en_ner_bionlp13cg_md']\n# models_dict = initilize_nlp_models(model_names)","execution_count":33,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"# Map entities existence in intro-result-discussion respectively to label values\nlabel_def = {'111':'prior-newdata','101':'prior-strong','100':'prior','001':'speculative','010':'unknown', '011':'novel'}\nchosen_models = list()\nchosen_models.append(generic_model)","execution_count":34,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"label_def_list = list(label_def.values())\n\nfor task,query in zip(tasks, queries):\n    label_entities = []\n    display(HTML(f'<h3 style=\"color:red\">Task - {task} \\n</h3>'))\n    numDocsFound, search_results = search(query)\n    if numDocsFound > 0:\n        task_answers = search_task_answers(search_results)\n        task_answers = populate_labels(task_answers)\n","execution_count":38,"outputs":[{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:red\">Task - Smoking and pre-existing pulmonary disease \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:blue\">Top 5 search result(s) of 2272 total. \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Journal Pre-proof Multi-omics Evaluation of Gastrointestinal and Other Clinical Characteristics of SARS-CoV-2 and COVID-19 Multi-omics Evaluation of Gastrointestinal and Other Clinical Characteristics of SARS-CoV-2 and COVID-19\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Since December 2019, coronavirus disease 2019 outbreak caused by severe acute respiratory syndrome coronavirus 2 has produced a worldwide panic.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. On the basis of prior evidence suggesting that ACE2 mediated the entry of SARS-CoV-2 into cells 2 , we previously found that ACE2 expression was significantly higher in smokers than in non-smokers, especially in distinct lung cell types 7 , corroborated by the clinical observation that the severe COVID-19 cases were more likely to have a smoking history than the non-severe COVID-19 1 .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Prevalence of Underlying Diseases in Hospitalized Patients with COVID-19: a Systematic Review and Meta-Analysis\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Severe disease may result in death due to progressive respiratory failure.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Although the results of the current analysis indicate that smoking can be an underlying factor that makes people susceptible to COVID-19 complications, in some studies, especially COVID-19 related studies, no strong evidence has been found regarding the correlation of COPD and smoking with being infected with this new virus.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. In summary, the results of the current study have shown that in patients with SARS-CoV-2 infection, hypertension, cardiovascular disease, smoking, and diabetes are the most prevalent co-existing disorders.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. - Smoking\n In the forest plot drawn, the pooled prevalence of SARS-CoV-2 infection in hospitalized patients with history of smoking was estimated as 7.63 percent.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. . \n- Chronic obstructive pulmonary diseases\n The last comorbidity that was studied using the included articles was chronic obstructive pulmonary diseases.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Respiratory infections in Norwegian dairy calves\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The frequency and severity of bovine respiratory infections have increased globally, and respiratory disease is currently regarded as the principal health problem and the most economically important disease in calves . The severity of respiratory disease seems to be influenced by immune status and the general condition of the animal, housing conditions, climate, management, and the presence and spreading of infectious agents. Respiratory viruses, which are effectively spread by aerosol and direct contact between animals and herds, usually act in combination with other infectious agents, in particular bacteria, in the development of respiratory disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. The incidence of respiratory disease has been found to peak in calves at between 5 and 7 wk of age . Respiratory diseases are often detected during autumn and winter , and are frequently associated with poorly ventilated housing conditions and drafts .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Nasal virome of dogs with respiratory infection signs include novel taupapillomaviruses\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Canine infectious respiratory disease is a major cause of morbidity and an important welfare issue for kennelled dog populations worldwide. The canine respiratory disease is generally characterized by coughing, snorting, wheezing, sneezing, nasal and eye discharge, fever, and dry and itchy nose.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. All of these viruses except canine circovirus and canine papillomavirus 9 have already been associated with canine respiratory disease although with different level of pathogenicity. Further studies will be required to determine the role of taupapillomaviruses in canine respiratory diseases. Such studies may include localizing the precise sites of viral replication in the canine respiratory track using in situ RNA hybridization and of their association with papillomatosis or other respiratory disease signs using matched case-control studies.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Detection of selected viral pathogens in dogs with canine infectious respiratory disease in Austria\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Canine infectious respiratory disease, synonymous for infectious tracheobronchitis or “kennel cough,” is a disease caused by single or multiple infectious agents with a high worldwide prevalence.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. However, even isolated viral infections may lead to clinically relevant and sometimes lethal respiratory disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Further studies from several countries detected CRCoV‐specific nucleic acid in dogs suffering from respiratory disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. Detection of further viral pathogens, pantropic canine coronavirus, canine hepacivirus and canine bocavirus) has been associated with respiratory disease in dogs.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. It was further investigated whether emerging viruses have a significantly higher prevalence in dogs with CIRD compared to dogs without respiratory disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:red\">Task - Co-infections, co-morbidities and respiratory infections \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:blue\">Top 5 search result(s) of 430 total. \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Two independent studies of infected populations in Wuhan, China found that approximately half the subjects infected with COVID-19 had an existing comorbidity . 2020), and in a study of 52 inpatients in Wuhan, 67% of patients with comorbidities died . However, the reason for this upregulation remains unclear, and a link to other COVID-19 comorbidities has not been determined. Here, we showed that ACE2 expression in lung tissue is upregulated by diseases representing comorbidities along with COVID-19.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. This identification of a common molecular mechanism of increased COVID-19 severity in patients with diverse comorbidities could direct the development of interventions to reduce the infection risk and disease severity in this population.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. To identify the genes highly associated with key comorbidities of severe COVID-19 .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>performed data analysis\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Since November 2019, the rapid outbreak of coronavirus disease 2019 , which arose from severe acute respiratory syndrome coronavirus 2 infection, has recently become a public health emergency of international concern .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Our study is the first nationwide investigation that systematically evaluates the impact of comorbidities on the clinical characteristics and prognosis in patients with COVID-19 in China. Circulatory and endocrine comorbidities were common among patients with COVID-19.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Comorbidities are present in around one fourth of patients with COVID-19 in China, and predispose to poorer clinical outcomes. A thorough assessment of comorbidities may help establish risk stratification of patients with COVID-19 upon hospital admission.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>History of Coronary Heart Disease Increases the Mortality Rate of Coronavirus Disease 2019 (COVID-19) Patients: A Nested Case- Control Study Based on Publicly Reported Confirmed Cases in Mainland China\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. In this paper, we used survival analysis to estimate the fatal risk of pre-existing comorbidities in COVID-19 in mainland China, adjusting for the confounding effect of age, gender, history of surgery, locations in and outside of Hubei Province, and early period of pandemic when no-intervention was taken. We first showed that a history of comorbidities significantly increased the death risk of COVID-19: one additional pre-existing comorbidity led to an estimated 29% increase of death risk.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Title Systematic review and meta-analysis of predictive symptoms and comorbidities for severe COVID-19 infection\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. In order to do this, a better understanding of the symptoms and comorbidities related to COVID-19 severity is required.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. The primary aim of this study is therefore to conduct a systematic review and meta-analysis, aggregating all currently available data from published studies, of symptoms and comorbidities predictive for severe illness with COVID-19.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Our univariable analysis is therefore valuable in evaluating specific individual symptoms and comorbidities predictive for COVID-19 severity using high-quality evidence in the form of peer-reviewed studies.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. The existing literature on COVID-19 fails to elucidate the specific symptoms and comorbidities most predictive for severe COVID-19 cases.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>European Journal of Case Reports in Internal Medicine Severe Immune Thrombocytopenia Complicated by Intracerebral Haemorrhage Associated with Coronavirus Infection: A Case Report and Literature Review\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The exact mechanism of ITP is poorly understood, with many hypotheses claiming that viral infection triggers the disease after which preformed antibodies cross-react with platelet antigens and is now considered to be associated with acute respiratory infections. Coronavirus\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:red\">Task - Neonates and pregnant women \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:blue\">Top 5 search result(s) of 27 total. \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Clinical features and obstetric and neonatal outcomes of pregnant patients with COVID-19 in Wuhan, China: a retrospective, single-centre, descriptive study\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The number of pregnant women with COVID-19 is also increasing; there were already more than 30 pregnant patients with COVID-19 in China by Feb 8, 2020 .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Up to now, seven liveborn infants have been delivered by pregnant women with COVID-19 at Tongji Hospital. We aim to describe epidemiological, clinical, laboratory, and radiological characteristics, treatment, and maternal and fetal outcomes of full-term pregnant women confirmed to have SARS-CoV-2 infection.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. COVID-19. After admission, the neonate had no fever and cough, with mild shortness of breath symptoms.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. This is a descriptive study on the clinical characteristics and obstetric and neonatal outcomes of pregnant women infected with COVID-19.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. This study is one of the first to report the maternal, fetal, and neonatal outcomes of pregnant women with COVID-19 who are in their third trimesters.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. In terms of clinical manifestations, the common symptoms of these pregnant women at the onset of COVID-19 were fever and cough, and the less common symptoms were diarrhoea and shortness of breath.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. First, only seven pregnant women with COVID-19 were included in the analyses.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Two patients had chronic diseases (hypothyroidism and polycystic ovary We assessed the incubation period, clinical manifestations, laboratory findings, chest CT scans, maternal and fetal management, therapeutic schedule, and maternal and fetal outcomes of full-term pregnant women confirmed to have COVID-19 pneumonia caused by SARS-CoV-2 infection.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Although one neonate was found to be infected with SARS-CoV-2 36 h postpartum, up to the follow-up day, no pneumonia and other clinical symptoms and signs were found in all seven neonates. To the best of our knowledge, we provide the first evidence that clinical features of COVID-19 in full-term pregnant women were similar to those of non-pregnant adult patients with COVID-19 and outcomes were good following intensive active management of the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Clinical characteristics and intrauterine vertical transmission potential of COVID-19 infection in nine pregnant women: a retrospective review of medical records\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The clinical characteristics of COVID-19 pneumonia in pregnant women were similar to those of non-pregnant adult patients with COVID-19 pneumonia.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. We report clinical data from nine pregnant women with laboratory-confirmed COVID-19 pneumonia.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. In the current study, we treated nine pregnant women with COVID-19 pneumonia in 11 days from Jan 20 to Jan 31, 2020.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. Future investigations of these issues and follow-up studies of pregnant women with COVID-19 infection, as well as neonates, will be necessary to ascertain the safety and health of mothers and babies exposed to SARS-CoV-2. In summary, the symptoms of pregnant women with COVID-19 pneumonia were diverse, with the main symptoms being fever and cough.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. We retrospectively reviewed clinical records, laboratory findings, and chest CT scans for nine pregnant women with laboratoryconfirmed COVID-19 pneumonia.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Data from laboratory tests showed that five of the nine pregnant women with COVID-19 pneumonia had lymphopenia.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Clinical features and the maternal and neonatal outcomes of pregnant women with coronavirus disease 2019\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The immunologic and physiologic changes might make pregnant women at higher risk of severe illness or mortality with Covid-19, compared with the general public.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. This study aimed to determine the clinical features and the maternal and neonatal outcomes of pregnant women with Covid-19.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. In this study, we reported a case series of pregnant women with Covid-19 and newborns born to these women.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. However, our data suggest the contrary to Covid-19. Among the 33 pregnant women with confirmed Covid-19, 39.4% of women presented mild, 57.6% presented moderate, and 3% presented with severe symptoms.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Compared with the general population, in which 5.0% who were admitted to the ICU, 2.3% who needed mechanical ventilation, and 1.4% who died 12 , it seems that the severity and mortality rate of Covid-19 in pregnant women are lower.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. Therefore, the obstetrical outcomes from pregnant women with SARS-CoV-2 infection appear better than that for pregnant women with SARS.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. However, there is currently no large series of pregnant women with confirmed Covid-19, so we believe that this study will be useful for guiding us in the management of pregnant women with Covid-19 and their newborns.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia: a case-control study\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. To date, none of previous studies have compared maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia to those without pneumonia, to investigate the adverse effects of COVID-19 infection on pregnancy.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Second, we collected the data of sixteen pregnant women with laboratory confirmed COVID-19 pneumonia and eighteen suspected cases with typical CT imaging. Although this is the largest number of pregnant women with COVID-19 pneumonia in literature so far, the sample size is still relatively small.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Why are pregnant women susceptible to COVID-19? An immunological viewpoint\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Moreover, due to the characteristic immune responses during pregnancy and potential risks from the cytokine-storm by COVID-19 infection, pregnant women with COVID-19 may face severe morbidity and even mortality.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Further studies are warranted to investigate the pregnant women with COVID-19 in the first and second trimester and follow-up the pregnancy outcomes and postnatal development of the fetus.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:red\">Task - Socio-economic and behavioral factors to understand the economic impact of the virus and whether there were differences. \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:blue\">Top 5 search result(s) of 4793 total. \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Investigating the Impact of Asymptomatic Carriers on COVID-19 Transmission\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The estimations of R 0 are expected to vary substantially per locality depending on how public health officials communicate the risk to the general public, general beliefs andinformation available to the population, and other socioeconomic and environmental factors affecting contact rates. Our goal with this investigation was to develop the SEYAR mean field estimate, which can be applied to different locations to provide a measure of the potential impact of the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Changes in behavioral patterns in response to an outbreak have an effect on the propagation of a disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Indeed, the cancellation of events likely to attract crowds, the closing of schools, and working from home will have a drastic impact on the size of the susceptible population at any given time.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>The effects of SNS communication: How expressing and receiving information predict MERS-preventive behavioral intentions in South Korea\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Despite this widespread social and economic panic, the South Korean government withheld important information such as the names of MERS-affected hospitals early in the outbreak.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The extent to which online content generation is gratifying is an important variable in determining its impact on general selfefficacy .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Interestingly, expressing MERS-related information had a direct negative relationship with MERSpreventive behavioral intentions.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. A second limitation concerns the measures of MERS-preventive behavioral intentions that were employed in our study. It is widely accepted that behavioral intentions are a significant predictor of actual behavior .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. Future studies need to employ actual preventive behavioral measures instead of preventive behavioral intentions as the main outcomes of communicating information via SNSs.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>\"The more I fear about COVID-19, the more I wear medical masks\": A survey on risk perception and medical masks' uses\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. However, there are growing bodies of examining the impact of COVID-19 outbreaks on different socioeconomic and political aspects in both a single country and the global scope.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Thirdly, price is the only factor that significantly decreases the willingness to buy medical masks.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Drawing a sample of 345 Vietnamese citizens from 15 to 47 years old, we found that the risk perception is the main factor which drives the behaviors in wearing medical masks in the public. This substantially contributes to the behavioral economics as well as infectious disease epidemiology to understand and creating behavior change to cope with this deadly virus. In addition, age is also a factor should be carefully examined later because we found that the elder is likely to wear masks after the COVID-19 outbreaks, with the weak evidence.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Perspectives of public health laboratories in emerging infectious diseases\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. To minimize the health and socioeconomic impacts of emerging epidemic infectious diseases, major challenges must be overcome in the national and international capacity for early detection, rapid and accurate etiological identification, rapid response and effective control.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Establishing laboratory and epidemiological capacity at the country and regional levels, therefore, is critical to minimize the impact of future emerging infectious disease epidemics.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Traditionally, emerging infectious diseases are broadly defined as infections that: have newly appeared in a population; are increasing in incidence or geographic range; or whose incidence threatens to increase in the near future.6,17 Six major factors, and combinations of these factors, have been reported to contribute to disease emergence and re-emergence: changes in human demographics and behavior; advances in technology and changes in industry practices; economic development and changes in land use patterns; dramatic increases in volume and speed of international travel and commerce; microbial mutation and adaptation; and inadequate public health capacity.6,17 From the perspective of public health planning and preparedness for effective emerging infectious disease surveillance, outbreak investigation and early response, the above working definition of emerging infectious disease and its associated factors that contribute to infectious disease emergence are too broad and generic for more specific application and for the development of a national public health system, especially in the context of a public health laboratory system in a country.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. The factors or combinations of factors contributing to the emergence of these pathogens also vary within each category.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. These four categories of emerging infectious diseases and the factors that contribute to the emergence of infectious diseases in each category are briefly described below.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Impacts of Social and Economic Factors on the Transmission of Coronavirus Disease 2019 (COVID-19) in China *\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Therefore, our estimated impacts have causal interpretations.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Furthermore, we examine the moderator effects of socioeconomic factors on the transmission of COVID-19 in China, which include population flow out of Wuhan, the distance between cities, GDP per capita, number of doctors, and contemporaneous weather characteristics.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. While many socioeconomic factors moderate the spread of the virus, the actual population flow from the source poses a higher risk to the destination than other factors such as geographic proximity and similarity in economic conditions.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. First, our analysis is connected to the economics and epidemiologic literature on the influencing factors of and ways of preventing the spread of infectious diseases.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. Besides, following the previous literature on the moderator effect of economic development and environmental characteristics on virus transmission, our analysis further reveal that COVID-19 transmission is positively moderated by GDP per capita and negatively moderated by the number of doctors, while the effects of the environmental factors and population density are mixed.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:red\">Task - Transmission dynamics of the virus, including the basic reproductive number, incubation period, serial interval, modes of transmission and environmental factors \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:blue\">Top 5 search result(s) of 7828 total. \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Modeling the Comparative Impact of Individual Quarantine vs. Active Monitoring of Contacts for the Mitigation of COVID-19\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Depending on how much time has passed since exposure to the primary infected individual, those infected may not yet be symptomatic -this period of time between infection and symptoms is an important epidemiological trait of an infectious disease called the incubation period.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Recent work on isolation for COVID-19 found a potentially large impact of perfect isolation, if one assumed there was limited presymptomatic transmission and a high probability of tracing contacts to be put under isolation immediately following symptom onset. 7 Our framework enables comparison of active monitoring and individual quarantine and considers parameters such as delays, and imperfect isolation to account for known nosocomial transmission of this respiratory virus. 8 One of the key uncertainties surrounding COVID-19 is the extent of asymptomatic and presymptomatic transmission. A recent study reporting asymptomatic transmission in Germany 9 was later found to be incorrect or misleading, 10 adding to the confusion. There has also been uncertainty about the serial interval -the time between symptom onset of infector-infectee pairs -which in turn reflects uncertainty about the extent of presymptomatic transmission.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Here, using methods described in previous work 6 and the latest epidemiological parameters reported for COVID-19, 11, 12 we compare the ability of individual quarantine and active monitoring to reduce the effective reproductive number of COVID-19 R e to below the critical threshold of one.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Under our fitted disease natural history parameters for serial interval scenario 1, with a shorter mean serial interval and hence substantial presymptomatic infectiousness, individual quarantine was considerably more effective than active monitoring at reducing onward transmission by an infected contact.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the The copyright holder for this preprint . days among 88 travelers 18 and a median of 4 days among 291 hospitalized study patients. 19 As a shorter incubation period relative to the serial interval would be consistent with less presymptomatic transmission, the incubation period distribution is a key parameter to refine as additional information becomes available. Under our fitted disease natural history parameters for serial interval scenario 2, with a mean serial interval of 7.5 days and hence a low amount of presymptomatic transmission, we found that both active monitoring and individual quarantine effectively reduced the expected number of secondary cases per contact below one.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Under both serial interval scenarios, in a low feasibility setting, the reproductive number was rarely brought below one under either individual quarantine or active monitoring, though a median reduction in the reproductive number of 21.0% and 13.6%, respectively for serial interval scenario 2, can still meaningfully slow the growth of an epidemic; for serial interval scenario 1, active monitoring resulted in only a 3% reduction in the reproductive number while individual quarantine resulted in a 17% reduction.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. Assuming a reported serial interval of mean 4.8 days, the incremental benefit of individual quarantine over active monitoring was substantial as a result of faster dynamics and more presymptomatic transmission.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. We fit the model assuming short versus long serial interval estimates. Model fitting by SMC for serial interval scenario 1 resulted in mean duration of infectiousness 2.4 days. Therefore, given the same incubation period distribution with mean = 5.2 days, a serial interval with mean = 4.8 days is best fit by substantial presymptomatic infectiousness while a longer serial interval with mean = 7.5 days is best fit by limited presymptomatic infectiousness.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. In locations at an early stage of COVID-19, where resources allow investment in high-quality implementation of interventions based on contact tracing, the effectiveness of individual quarantine or active monitoring depends heavily on i) aspects of the disease, especially the assumed serial interval and timing of presymptomatic transmission, and ii) aspects of the setting including the fraction of contacts traced. Briefly, a shorter serial interval, larger window of presymptomatic transmission, poor quality interventions, and a small fraction of contacts traced all reduce the ability of either intervention to decrease transmission.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Transmission interval estimates suggest pre-symptomatic spread of COVID-19\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Here we use transmission clusters in two locations where cases have reported links, exposure and symptom onset times to estimate both the incubation period and serial interval of COVID-19.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Furthermore, we estimate the serial interval to be shorter than the incubation period in both clusters, which suggests pre-symptomatic transmission.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. In conclusion, in both the Singapore and Tianjin COVID-19 clusters we identified both the incubation period and the serial interval. Our results suggest that there is substantial pre-symptomatic transmission, with the serial interval shorter than incubation period by 2-4 days.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. Finally, our shorter serial intervals lead to an estimate of the basic reproduction number of approximately 2 in both datasets, suggesting that stopping half the transmission events may be sufficient to control outbreaks. 14 .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Estimation of the Basic Reproductive Number and Mean Serial Interval of a Novel Pathogen in a Small, Well-Observed Discrete Population\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. This uncertainty is reduced greatly once the basic reproductive number and the serial interval of infection are known. The basic reproductive number quantifies the transmissibility of a pathogen and is defined as the average number of secondary infections generated by one typically infectious individual in an otherwise susceptible population. The serial interval in each infector-infectee pair is the time between disease onset in the infector and the onset in the infectee. In this study, we present a temporal likelihood model that allows real-time simultaneous estimation of R0 and the average of serial interval Tg in a small well-observed population, so that it provides timely information for informed public health responses.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. We have described a quantitative method that allows the rapid joint estimation of the basic reproductive number, and mean and variance of serial interval for an emerging pathogen based only on case data from an outbreak in a small population.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. In essence, this approach is able to tease apart the two parameters from case data because it extracts key information from the transmission dynamics of different phases of a respiratory epidemic: the initial stochastic period of growth, the deterministic period of exponential growth, the sub-exponential period of growth prior to peak incidence, and the peak itself.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. We have also assumed throughout that offspring distribution and serial interval distribution follow a Poisson density.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. However, the ability of these methods to independently estimate both means and variances of the offspring distribution and serial interval distribution remains to be described. We would anticipate that the estimation of variance for the serial interval distribution would be challenging for cases other than where there are very high volumes of data for the first few generations or when the true distribution has a low variance.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Although the pattern of univariate inference for Tg was similar to that for R0, the final confidence intervals were wider than for R0.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. However, prediction intervals were wider.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. The estimates of different values of k were tabulated on Table 1 for the model using Poisson-distributed serial interval for the estimates on the last day of observation using different numbers of iterations: 10, 20, 100, along with different k values.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Estimation of the Basic Reproductive Number and Mean Serial Interval of a Novel Pathogen in a Small, Well-Observed Discrete Population\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. This uncertainty is reduced greatly once the basic reproductive number and the serial interval of infection are known. The basic reproductive number quantifies the transmissibility of a pathogen and is defined as the average number of secondary infections generated by one typically infectious individual in an otherwise susceptible population. The serial interval in each infector-infectee pair is the time between disease onset in the infector and the onset in the infectee. In this study, we present a temporal likelihood model that allows real-time simultaneous estimation of R 0 and the average of serial interval T g in a small well-observed population, so that it provides timely information for informed public health responses.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. We have described a quantitative method that allows the rapid joint estimation of the basic reproductive number, and mean and variance of serial interval for an emerging pathogen based only on case data from an outbreak in a small population.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. In essence, this approach is able to tease apart the two parameters from case data because it extracts key information from the transmission dynamics of different phases of a respiratory epidemic: the initial stochastic period of growth, the deterministic period of exponential growth, the sub-exponential period of growth prior to peak incidence, and the peak itself.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. We have also assumed throughout that offspring distribution and serial interval distribution follow a Poisson density.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. Univariate and bivariate estimates of the model using Poisson distributed serial interval, different k values, and number of simulation iterations.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. However, the ability of these methods to independently estimate both means and variances of the offspring distribution and serial interval distribution remains to be described.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Although the pattern of univariate inference for T g was similar to that for R 0 , the final confidence intervals were wider than for R 0 .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. However, prediction intervals were wider.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. The estimates of different values of k were tabulated on Table 1 for the model using Poisson-distributed serial interval for the estimates on the last day of observation using different numbers of iterations: 10, 20, 100, along with different k values.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Evaluating new evidence in the early dynamics of the novel coronavirus COVID-19 outbreak in Wuhan, China with real time domestic traffic and potential asymptomatic transmissions\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. A recent concern is the potential presence of asymptomatic transmission. One confirmed case in Germany suggests that the infection may have been transmitted during the incubation period of the index patient . In the dynamics of an outbreak, the basic reproductive number R 0 is an important epidemiological parameter: it measures the number of cases an initial infectious individual can generate in a fully susceptible population.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. With these concerns, Wu et al. and Nishiura, Jung, et al. used an alternative data source, i.e., the number of internationally exported cases between Dec. 31, 2019 and Jan. 28, 2020, to infer the disease dynamics.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. In this study, the author modifies the SEIR model to evaluate the empirical support for the presence of asymptomatic transmissions based on current evidence.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The updated basic reproductive number was found to be 2.12 on average, which is significantly smaller than previous estimates , probably because the new estimate factors in the social and non-pharmaceutical mitigation implemented in Wuhan through the evacuee dataset.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Increasing the outward flow effectively reduces the proportion infected and subsequently the basic reproductive number.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. With a 15% increase in the international outbound flow, sensitivity analysis shows a c. 2% decrease in the estimated basic reproductive number.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. Reducing the zoonotic infection force or shortening its duration has the effect of increasing the estimate of the basic reproductive number, and vice versa.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. However, it is known that the basic reproductive number vary with human social behavior and organization .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. During this period, the overall outflow was 15% more than the same period in 2019.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Technically, the SEAIR model is not statistically significant at the 5% level; however, due to the relatively small sample size of the infection data with respect to the number of residents in Wuhan, it is inconclusive whether the disease dynamics show evidence for the SEAIR model.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. The SEIR model produced an R 0 with a mean of 2.12 and a 95% credible interval author/funder, who has granted medRxiv a license to display the preprint in perpetuity. is the The copyright holder for this preprint . estimated to be on average 37,115 individuals with a 95% credible interval of.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:red\">Task - Severity of disease, including risk of fatality among symptomatic hospitalized patients, and high-risk patient groups \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:blue\">Top 5 search result(s) of 8724 total. \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Knowledge and awareness of Middle East respiratory syndrome coronavirus among Saudi and Non-Saudi Arabian pilgrims Original Article\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Although the disease was detected in about 27 countries, >80% of cases were detected in Saudi Arabia. 2 MERS-CoV is classified as lineage C of the genus Betacoronavirus. 3 Insecteating bats are currently the known potential reservoir host for MERS-CoV 4,5 while dromedary camels are the known potential animal host. 6, 7 Consumption of raw camel products and direct contact with camels may constitute a possible role of disease transmission. 8 MERS-CoV affects all ages. The clinical epidemiology of MERS-CoV cases showed similarity to the seasonal flu; however, severe cases have been reported in elderly individuals and immune compromised patients with underlying conditions. 9 Health-care workers are among the risk groups 2 and asymptomatic health-care workers spread illness among hospitalized patients. 10 Annually, Saudi Arabia hosts at least three million Muslims from different nationalities at the same time in Mekkah during the pilgrimage season.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Accordingly, there is a possibility of transmission of different diseases among pilgrims.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Appropriate public health responses are dependent on highquality knowledge and awareness about important aspects International Journal of Health Sciences Vol. 11, Issue 5 of the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. We argue that our thinking should converge on the public awareness and we should address the gaps of all disciplines of the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The Saudi pilgrims showed a good level of the overall knowledge about the nature of MERS-CoV including: The causative agent, the signs of the disease, its similarity to flu, the severity of the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. In contrast, both groups showed a low level of knowledge regarding the seasonal nature of the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. This finding agrees with the results reported previously in Tashani et al. 2014. 19 Health-care workers are prone to capture infection from their close contact with infected individuals, accordingly, they are considered among the risk groups. 2 In addition, asymptomatic health-care workers play an important role in spreading the illness among hospitalized patients. 10 The Saudi pilgrims showed a higher level of knowledge regarding the fact that being health-care workers is a risk factor.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. The knowledge and awareness about the disease are important for the adoption of the protective measures that minimize the risk exposure for the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. However, in both groups misperceptions toward high-risk believes still exist.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Both groups of pilgrims were first screened for the overall knowledge and awareness toward MERS-CoV.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. The Saudi pilgrims showed significant differences in the level of knowledge than non-Saudi Arabian pilgrims in the following topics: The nature of MERS-CoV, the signs of the disease and its similarity to flu, the severity of the disease and the animals that could transmit the disease, . Both groups showed a low level of knowledge regarding the seasonal nature of the disease, and camel milk and meat as possible sources of disease transmission . No significant differences were detected among male and female participants in both groups.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. No significant correlation was detected among different questions.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Knowledge and awareness of Middle East respiratory syndrome coronavirus among Saudi and Non-Saudi Arabian pilgrims\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Although the disease was detected in about 27 countries, >80% of cases were detected in Saudi Arabia.2 MERS-CoV is classified as lineage C of the genus Betacoronavirus.3 Insect-eating bats are currently the known potential reservoir host for MERS-CoV4,5 while dromedary camels are the known potential animal host.6,7 Consumption of raw camel products and direct contact with camels may constitute a possible role of disease transmission.8 MERS-CoV affects all ages. The clinical epidemiology of MERS-CoV cases showed similarity to the seasonal flu; however, severe cases have been reported in elderly individuals and immune compromised patients with underlying conditions.9 Health-care workers are among the risk groups2 and asymptomatic health-care workers spread illness among hospitalized patients.10 Annually, Saudi Arabia hosts at least three million Muslims from different nationalities at the same time in Mekkah during the pilgrimage season.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Accordingly, there is a possibility of transmission of different diseases among pilgrims. Interestingly, no notification of laboratory-confirmed MERS-CoV cases during pilgrimage seasons was reported; however, several imported MERS-CoV cases were recorded among returning individuals who visited Mekkah and Medinah in different countries including the Netherlands, UK, Algeria, Tunisia, and Malaysia.11-13 Appropriate public health responses are dependent on high-quality knowledge and awareness about important aspects of the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. We argue that our thinking should converge on the public awareness and we should address the gaps of all disciplines of the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The Saudi pilgrims showed a good level of the overall knowledge about the nature of MERS-CoV including: The causative agent, the signs of the disease, its similarity to flu, the severity of the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. In contrast, both groups showed a low level of knowledge regarding the seasonal nature of the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. This finding agrees with the results reported previously in Tashani et al. 2014.19 Health-care workers are prone to capture infection from their close contact with infected individuals, accordingly, they are considered among the risk groups.2 In addition, asymptomatic health-care workers play an important role in spreading the illness among hospitalized patients.10 The Saudi pilgrims showed a higher level of knowledge regarding the fact that being health-care workers is a risk factor.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. The knowledge and awareness about the disease are important for the adoption of the protective measures that minimize the risk exposure for the disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. However, in both groups misperceptions toward high-risk believes still exist.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Both groups of pilgrims were first screened for the overall knowledge and awareness toward MERS-CoV.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. The Saudi pilgrims showed significant differences in the level of knowledge than non-Saudi Arabian pilgrims in the following topics: The nature of MERS-CoV, the signs of the disease and its similarity to flu, the severity of the disease and the animals that could transmit the disease,. Both groups showed a low level of knowledge regarding the seasonal nature of the disease, and camel milk and meat as possible sources of disease transmission. No significant differences were detected among male and female participants in both groups.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. No significant correlation was detected among different questions.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Journal Pre-proof Asymptomatic carrier state, acute respiratory disease, and pneumonia due to severe acute respiratory syndrome coronavirus 2 (SARSCoV-2): Facts and myths\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The pathogen of this disease was confirmed as a novel coronavirus by molecular methods and was initially named as 2019 novel coronavirus; however, the World Health Organization announced a new name on February 11, 2020 for the epidemic disease: Corona Virus Disease .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. In addition, the Coronavirus Study Group of the International Committee on Taxonomy of Viruses has renamed the virus, which was provisionally named 2019-nCoV, as severe acute respiratory syndrome coronavirus-2 based on phylogeny, taxonomy, and established practice. 5 Prior to this, on January 30, 2020, the WHO declared the COVID-19 outbreak as the sixth public health emergency of international concern; therefore, this outbreak constitutes a public health risk through the international spread of disease and requires a coordinated international response.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Therefore, herein, we performed a literature review, focusing on the epidemiological characteristics and clinical manifestations of COVID-19, including asymptomatic carrier state, acute respiratory disease, and pneumonia.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Although the risk factors of COVID-19 remain unclear, many studies reported that a significant proportion of patients had underlying conditions. 8 Finally, at least 20% of COVID-19 cases had underlying diseases, and more severe cases were more likely to have comorbidities than non-severe cases.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Initial studies reported that the mortality rate associated with SARS-CoV-2 pneumonia ranged from 11% 12 to 15%, 13 but later studies revealed that the mortality rate was between 1.4% 11 and 4.3%. 14 The differences in the results among different studies could be due to the study population, as well as the differences among the studies in terms of disease severity.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. In one study using the MulBSTA score system, 46 which includes six indices, namely multilocular infiltration, lymphopenia, bacterial co-infection, smoking history, hypertension, and age, revealed that these indices were poor prognostic factors. 12 Another study showed similar findings, and specifically the indicators of disease severity, including oxygenation, respiratory rate, leukocyte/lymphocyte count, and the chest imaging findings, were associated with a poor clinical outcome. 11 Moreover, a substantially 18 elevated case-fatality rate included the following patient characteristics: male sex, ≥ 60 years of age, baseline diagnosis of severe pneumonia, and delay in diagnosis. 15 Similarly, the China CDC reported that patients aged ≥ 80 years had the highest case fatality rate, 14.8%, among different age groups, and the case fatality rate of patients in which disease severity was critical was 49.0%. 8 Together, these findings suggest that old age and increased disease severity could predict a poor outcome.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. More studies are needed to clarify the epidemiologic characteristics of COVID-19 and to identify the risk and prognostic factors of patients infected with SARS-CoV-2. Second, Zou et al. reported that the viral load detected in asymptomatic patients was similar to that found in symptomatic patients; however, the viral loads from patients with severe diseases were higher than those in patients with mild-to-moderate presentations.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Journal Pre-proof Asymptomatic carrier state, acute respiratory disease, and pneumonia due to severe acute respiratory syndrome coronavirus 2 (SARSCoV-2): Facts and myths\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The pathogen of this disease was confirmed as a novel coronavirus by molecular methods and was initially named as 2019 novel coronavirus; however, the World Health Organization announced a new name on February 11, 2020 for the epidemic disease: Corona Virus Disease .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. In addition, the Coronavirus Study Group of the International Committee on Taxonomy of Viruses has renamed the virus, which was provisionally named 2019-nCoV, as severe acute respiratory syndrome coronavirus-2 based on phylogeny, taxonomy, and established practice. 5 Prior to this, on January 30, 2020, the WHO declared the COVID-19 outbreak as the sixth public health emergency of international concern; therefore, this outbreak constitutes a public health risk through the international spread of disease and requires a coordinated international response.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Therefore, herein, we performed a literature review, focusing on the epidemiological characteristics and clinical manifestations of COVID-19, including asymptomatic carrier state, acute respiratory disease, and pneumonia.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Although the risk factors of COVID-19 remain unclear, many studies reported that a significant proportion of patients had underlying conditions. 8 Finally, at least 20% of COVID-19 cases had underlying diseases, and more severe cases were more likely to have comorbidities than non-severe cases.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Initial studies reported that the mortality rate associated with SARS-CoV-2 pneumonia ranged from 11% 12 to 15%, 13 but later studies revealed that the mortality rate was between 1.4% 11 and 4.3%. 14 The differences in the results among different studies could be due to the study population, as well as the differences among the studies in terms of disease severity.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. In one study using the MulBSTA score system, 46 which includes six indices, namely multilocular infiltration, lymphopenia, bacterial co-infection, smoking history, hypertension, and age, revealed that these indices were poor prognostic factors. 12 Another study showed similar findings, and specifically the indicators of disease severity, including oxygenation, respiratory rate, leukocyte/lymphocyte count, and the chest imaging findings, were associated with a poor clinical outcome. 11 Moreover, a substantially 18 elevated case-fatality rate included the following patient characteristics: male sex, ≥ 60 years of age, baseline diagnosis of severe pneumonia, and delay in diagnosis. 15 Similarly, the China CDC reported that patients aged ≥ 80 years had the highest case fatality rate, 14.8%, among different age groups, and the case fatality rate of patients in which disease severity was critical was 49.0%. 8 Together, these findings suggest that old age and increased disease severity could predict a poor outcome.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. More studies are needed to clarify the epidemiologic characteristics of COVID-19 and to identify the risk and prognostic factors of patients infected with SARS-CoV-2. Second, Zou et al. reported that the viral load detected in asymptomatic patients was similar to that found in symptomatic patients; however, the viral loads from patients with severe diseases were higher than those in patients with mild-to-moderate presentations.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>MISSING\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Some studies have found higher virus load for influenza viruses among patients who were sicker . Higher virus loads of human bocavirus in nasopharyngeal aspirate samples were associated with greater severity of illness among Chinese children .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Because the severity of patients most likely was associated with the available specimen, we restricted our analysis to patients with nasopharyngeal swab samples, which is the recommended specimen for initial MERS-CoV diagnosis according to KSA MOH guidelines .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Although several studies have shown underlying illness to be common among MERS-CoV patients, including those who died, few have quantified the risk for underlying illness in severe outcomes .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. Third, data for other outcomes indicative of severity of illness were either missing in some patients or not available at all in the current dataset.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The factors dictating severity of illness and outcome among MERS-CoV patients are still not well defined.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Using the composite severity variable, we categorized 48 illnesses as severe; for these patients, Ct values were significantly lower than for the 54 patients without severe disease . Sixty-seven patients were hospitalized. C t values were significantly lower for the 36 hospitalized patients admitted to the ICU than for the 31 hospitalized patients admitted the general ward. Of the 61 patients who survived, Ct values were not significantly lower for the 30 symptomatic patients than they were for the 31 mildly symptomatic or asymptomatic patients .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:red\">Task - Susceptibility of populations \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:blue\">Top 5 search result(s) of 3391 total. \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Estimation of country-level basic reproductive ratios for novel Coronavirus (COVID-19) using synthetic contact matrices\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Here we use the estimated social contact matrix for China to define the early age-structured transmission dynamics; this is formed from the country-specific social contact matrix k and a vector of age-specific susceptibilities z -whose entries scale the risk of transmission from contact with infected individuals to susceptible individuals of each age class.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Here we have developed a simple model for age-structured transmission of 2019-nCoV with two components: an age-structured contact matrix dependent on the behaviour of the host population and an age-dependent susceptibility profile dependent on physiological response to infection. By using a previously-estimated synthetic contact matrix and age-stratified data, we were able to estimate age-dependent susceptibility profiles based on the first 425 and the first 4,021 cases in China.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. We demonstrated that taking age-specific susceptibility into account results in substantially different predictions of transmission intensity by country relative to a model without age-specific susceptibility; countries with older populations are at substantially higher risk than countries with younger populations.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Secondly, in inferring the age-dependent susceptibility, we are effectively generating a matrix R i,j which determines the distribution of secondary confirmed cases in terms of current confirmed cases.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. In Figure 1 we present the results of a null-model in which there is no age-specific susceptibility, and plot the associated scaling factor σ for each of the 152 countries included in Prem et al.'\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. In Figure 3 we plot the scaling factors obtained through our consideration of age-dependent susceptibility against the factors obtained by assuming homogeneous susceptibility across age groups.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. When we incorporate age-dependent susceptibility, contact patterns involving members of highly susceptible age classes becomes particularly important -generating a 3 .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. For COVID-19, this means that we see higher scaling factors in countries with older populations, since this usually means contacts involving older individuals are more common.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. However, the maps in Figure 2 demonstrate that when age-specific susceptibility is taken into account, the pattern of infectious potential by country is generally reversed.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Association of functional polymorphisms in the MxA gene with susceptibility to enterovirus 71 infection\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. We examined whether the MxA polymorphisms were related to the risk and severity of enterovirus 71 infection in Chinese populations.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Significantly decreased susceptibility to EV71 infection was observed for the -123A allele and -88T allele carriers, with ORs estimated as 0.56 and 0.64, respectively, in the 1 3 family.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Individual susceptibility to EV71 infection seems to be variable to severe neurological disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. The identification of susceptibility genes contributing to EV71 infection would assist in predicting individual and population risks of EV71 infection, as well as clarifying pathogenesis of this infectious disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. In the present study, we examined whether polymorphisms in the MxA gene were related to the risk and severity of EV71 infection among Chinese populations.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. This result may be due to the limited sample size of patients with uncomplicated cases in the southern population in our study.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. To our knowledge, only three genetic association studies have addressed susceptibility genes that related to EV71 infection.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Compared to the complicated cases, the uncomplicated cases had a comparable MxA transcription level subsequent studies in other populations.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. Recently, several association studies have shown that the MxA gene was related to the susceptibility to various specific infections .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. Although the highly significant association between MxA and susceptibility to EV71 derived from a biologically based a priori hypothesis, our findings should be independently verified in populations of different ancestry, such as white subjects and African subjects.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. The Mantel-Haenszel test was performed for the combined analysis of the data from both populations.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Association of functional polymorphisms in the MxA gene with susceptibility to enterovirus 71 infection\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Individual susceptibility to EV71 infection seems to be variable.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. The identification of susceptibility genes contributing to EV71 infection would assist in predicting individual and population risks of EV71 infection, as well as clarifying pathogenesis of this infectious disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. On the basis of the functional role of MxA in the pathogenesis of viral infections in vivo and in vitro, we hypothesize that MxA might be a potential candidate susceptibility gene for EV71 infection.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. In the present study, we examined whether polymorphisms in the MxA gene were related to the risk and severity of EV71 infection among Chinese populations.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. This result may be due to the limited sample size of patients with uncomplicated cases in the southern population in our study.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. To our knowledge, only three genetic association studies have addressed susceptibility genes that related to EV71 infection. Some of the results, however, could not be replicated in subsequent studies in other populations.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Recently, several association studies have shown that the MxA gene was related to the susceptibility to various specific infections.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. Although the highly significant association between MxA and susceptibility to EV71 derived from a biologically based a priori hypothesis, our findings should be independently verified in populations of different ancestry, such as white subjects and African subjects.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Haplotypic diversity in human CEACAM genes: effects on susceptibility to meningococcal disease\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Adhesion of Neisseria meningitidis to the human nasopharyngeal epithelium is critical for establishment of invasive meningococcal disease, 1 a globally important infection that cannot be effectively controlled due to the lack of a comprehensive vaccine. 2 In Europe 1-6/100 000 people contract the disease annually, with a case fatality rate of approximately 8%. 3 Asymptomatic carriage is more frequent however, with meningococci present in approximately 10% of the general population and up to 40% in 15-24 year olds. 4 Both colonization and invasion require the specific interaction of Opacity-associated adhesin proteins on the meningococcal surface with the carcino-embryonic antigen cell adhesion molecule family of cell surface proteins on the human nasopharyngeal epithelium. 5 Most individuals are colonized asymptomatically but, for incompletely understood reasons, adhesion in some hosts is followed by invasion of the mucosa, entry of meningococci into the blood stream and meningococcal disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Host genetics is thought to contribute one-third of meningococcal disease susceptibility 18 but little is known about the population genetic diversity of the CEACAM family and its influence on susceptibility, which may be manifested at the level of interaction between bacterium and host. Genetic diversity in CEACAM may result in particular members of the human population being more easily colonized by meningococci, perhaps placing them at an increased risk of invasive disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. This study was undertaken to determine whether genetic differences in human CEACAM genes are associated with susceptibility to meningococcal disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Here we describe the genotypic diversity of four CEACAM genes in a case/control study cohort of Caucasians to determine whether the diversity in these genes influences susceptibility to meningococcal disease. Three haplotypes in two CEACAM genes were found to affect susceptibility, which may exert their effects through two mechanisms.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. CEACAM diversity may influence human susceptibility to meningococcal disease in two ways.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. In this study, there was no evidence of polymorphism in the Opa binding site, encoded by the first and second exons of each CEACAM gene, affecting disease susceptibility.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. A number of examples of polymorphisms in cell surface receptor-encoding genes affecting human disease susceptibility to a variety of pathogens have been reported.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Haplotypic diversity in human CEACAM genes: effects on susceptibility to meningococcal disease\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Adhesion of Neisseria meningitidis to the human nasopharyngeal epithelium is critical for establishment of invasive meningococcal disease,1 a globally important infection that cannot be effectively controlled due to the lack of a comprehensive vaccine.2 In Europe 1–6/100 000 people contract the disease annually, with a case fatality rate of approximately 8%.3 Asymptomatic carriage is more frequent however, with meningococci present in approximately 10% of the general population and up to 40% in 15–24 year olds.4 Both colonization and invasion require the specific interaction of Opacity-associated adhesin proteins on the meningococcal surface with the carcino-embryonic antigen cell adhesion molecule family of cell surface proteins on the human nasopharyngeal epithelium.5 Most individuals are colonized asymptomatically but, for incompletely understood reasons, adhesion in some hosts is followed by invasion of the mucosa, entry of meningococci into the blood stream and meningococcal disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Host genetics is thought to contribute one-third of meningococcal disease susceptibility18 but little is known about the population genetic diversity of the CEACAM family and its influence on susceptibility, which may be manifested at the level of interaction between bacterium and host. Genetic diversity in CEACAM may result in particular members of the human population being more easily colonized by meningococci, perhaps placing them at an increased risk of invasive disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. This study was undertaken to determine whether genetic differences in human CEACAM genes are associated with susceptibility to meningococcal disease.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. Here we describe the genotypic diversity of four CEACAM genes in a case/control study cohort of Caucasians to determine whether the diversity in these genes influences susceptibility to meningococcal disease. Three haplotypes in two CEACAM genes were found to affect susceptibility, which may exert their effects through two mechanisms.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. CEACAM diversity may influence human susceptibility to meningococcal disease in two ways.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. In this study, there was no evidence of polymorphism in the Opa binding site, encoded by the first and second exons of each CEACAM gene, affecting disease susceptibility.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. A number of examples of polymorphisms in cell surface receptor-encoding genes affecting human disease susceptibility to a variety of pathogens have been reported.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:red\">Task - Public health mitigation measures that could be effective for control \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h3 style=\"color:blue\">Top 5 search result(s) of 6576 total. \n</h3>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Public preferences for interventions to prevent emerging infectious disease threats: a discrete choice experiment\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. In the aftermath of the 2009 influenza pandemic, particularly severe criticism was voiced from some European quarters that health authorities and WHO over-reacted in their response to the influenza A pandemic of 2009.25 Others argued that the prior experiences with SARS-CoV and H5N1 justified a robust response to the influenza A pandemic in 2009.22 23 For instance, in Singapore, where SARS-CoV caused 238 infections, 33 deaths and reduced annual GDP by 1%,26 there was a strong governmental response, with over 8000 contacts of SARS-CoV cases quarantined.27 Singapore’s health authorities used similarly robust measures during the influenza pandemic in 2009, and initially mandated isolation of confirmed cases, quarantining of exposed individuals, and temperature screening in hospitals, public places and ports of entry.28 This was in spite of the paucity of evidence that such measures would make any difference to the overall course of the pandemic. Since public acceptance is vital for many outbreak response policies to have the intended effect, public health decision making must go beyond accurate assessments of the severity of a threat and what measures might be effective against it.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. This framework must be able to accommodate a diversity of scenarios on the key characteristics of the threat, and the suite of public health interventions available. Public health policy makers have several constituencies, but chief among these is the populace on whose behalf they make decisions.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. This gap is especially noteworthy due to the considerable personal burden imposed by some interventions and the backlash against what some deemed excessive measures during the 2009 influenza pandemic.25 30\n To this end, we conducted a ‘conjoint’ discrete choice experiment to explore the Singapore public’s willingness to accept certain containment and mitigation strategies under various outbreak scenarios, spanning a range of infection and fatality rates.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. It is not clear how effective social distancing measures would be in a mild outbreak, however, as many respondents reported that they would not voluntarily reduce how frequently they go out to eat or shop.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Studies such as this one can also identify misperceptions among the public that may hinder support for effective control.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Future studies might evaluate the effect of priming respondents with accurate public health information before eliciting their preferences.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. It has been argued that, during the influenza pandemic in 2009, the failure of WHO and government health agencies was not so much in their assessment of the severity of the threat or what interventions might be effective against it, but in being able to effectively communicate the risk and the benefits of select mitigation strategies.60 61 Prior assessment of population level preferences could help identify thresholds of epidemic severity which could serve as triggers for the implementation of more disruptive measures.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. Similar studies in other settings would be useful for public health decision makers.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Personal View Can we contain the COVID-19 outbreak with the same measures as for SARS?\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. By Jan 30, 2020, COVID19 was declared a public health emergency of international concern.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. However, the similarities end here. successful and included active case detection, isolation of cases, contact tracing and quarantine of all contacts, social distancing, and community quarantine. • Whether these measures will also be successful for COVID-19 will not depend on the similarities but the differences between SARS and COVID-19. • Clear differences are emerging, such as in transmissibility and severity pyramids; COVID-19 has a higher transmissibility than SARS, and many more patients with COVID-19 rather than SARS have mild symptoms that contribute to spread because these patients are often missed and not isolated. • Because of the extent of community spread, traditional public health measures might not be able to halt all human-to-human transmission, and we need to consider moving from containment to mitigation.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. China should be commended for its political will in implementing what might appear to be extreme measures.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. However, closures of institutions and public places, and restrictions in travel and trade, cannot be maintained indefinitely. Countries have to face the reality that individualcase containment might not be possible in the long run, and there might be the need to move from containment to mitigation, balancing the costs and benefits of public health measures. Even if our public health measures are not able to fully contain the spread of COVID19 because of the virus characteristics, they will still be effective in delaying the onset of widespread community trans mission, reducing peak incidence and its impact on public services, and decreasing the overall attack rate. In addition, minimising the size of the outbreak or suppressing its peak can reduce global deaths by providing health systems with the opportunity to scale up and respond, and to slow down the global spread until effective vaccines become available.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Suppression and Mitigation Strategies for Control of COVID-19 in New Zealand\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. This means that without controls there is likely to be widespread infection, which may overwhelm health care systems and lead to large numbers of deaths. In this study we examined potential control scenarios generated by a standard mathematical modelling approach to epidemic spread to consider the impacts on the progression of the disease in New Zealand.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. In addition, successful mitigation requires periods of these intensive control measures to be continued for up to 2.5 years before the population acquires a sufficient level of herd immunity.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Furthermore, correct timing of strong control measures is crucial to successfully keeping healthcare systems from being overloaded.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Suppression strategies aim to keep the number infections to a minimum for as long as possible, by early instigation of control measures.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. If strong suppression is successful in reducing the number of cases close to zero, it is possible that some control measures could be lifted.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. Figure 5 . Mitigation strategies in which strong control fails to reduce the effective reproduction number Rc below 1.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Public preferences for interventions to prevent emerging infectious disease threats: a discrete choice experiment\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. In the aftermath of the 2009 influenza pandemic, particularly severe criticism was voiced from some European quarters that health authorities and WHO over-reacted in their response to the influenza A pandemic of 2009. 25 Others argued that the prior experiences with SARS-CoV and H5N1 justified a robust response to the influenza A pandemic in 2009. 22 23 For instance, in Singapore, where SARS-CoV caused 238 infections, 33 deaths and reduced annual GDP by 1%, 26 there was a strong governmental response, with over 8000 contacts of SARS-CoV cases quarantined. 27 Singapore's health authorities used similarly robust measures during the influenza pandemic in 2009, and initially mandated isolation of confirmed cases, quarantining of exposed individuals, and temperature screening in hospitals, public places and ports of entry. 28 This was in spite of the paucity of evidence that such measures would make any difference to the overall course of the pandemic. Since public acceptance is vital for many outbreak response policies to have the intended effect, public health decision making must go beyond accurate assessments of the severity of a threat and what measures might be effective against it.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. This framework must be able to accommodate a diversity of scenarios on the key characteristics of the threat, and the suite of public health interventions available. Public health policy makers have several constituencies, but chief among these is the populace on whose behalf they make decisions.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. This gap is especially noteworthy due to the considerable personal burden imposed by some interventions and the backlash against what some deemed excessive measures during the 2009 influenza pandemic. 25 30 To this end, we conducted a 'conjoint' discrete choice experiment to explore the Singapore public's willingness to accept certain containment and mitigation strategies under various outbreak scenarios, spanning a range of infection and fatality rates.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>discussion\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. It is not clear how effective social distancing measures would be in a mild outbreak, however, as many respondents reported that they would not voluntarily reduce how frequently they go out to eat or shop.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. Studies such as this one can also identify misperceptions among the public that may hinder support for effective control.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. Future studies might evaluate the effect of priming respondents with accurate public health information before eliciting their preferences.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. It has been argued that, during the influenza pandemic in 2009, the failure of WHO and government health agencies was not so much in their assessment of the severity of the threat or what interventions might be effective against it, but in being able to effectively communicate the risk and the benefits of select mitigation strategies. 60 61 Prior assessment of population level preferences could help identify thresholds of epidemic severity which could serve as triggers for the implementation of more disruptive measures.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. Similar studies in other settings would be useful for public health decision makers.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<h4><i>Lessons drawn from China and South Korea for managing COVID-19 epidemic: insights from a comparative modeling study\n</i></h4>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>introduction\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. In the absence of effective treatments and vaccines, an early adoption of stringent public health measures is crucial in mitigating the scale and burden of an outbreak.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. However, these public health interventions may not be considered and/or implemented as effectively in other settings and contexts.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>3. For instance, particularly restrictive measures may not be effective in certain countries 6 .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>4. As such, it is not recommended for implementation, if not within a broader package of public health measures aimed at rapidly containing the outbreak 8 .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>5. This led to rapid and effective mitigation of the COVID-19 epidemic.\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"\t<h5>result\n</h5>\n"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>1. We obtained data of the confirmed COVID-19 cases, cumulative number of quarantined individuals, cumulative death cases in mainland China from the \"National Health Commission\" of the People's Republic of China 14 .\n</blockquote>"},"metadata":{}},{"output_type":"display_data","data":{"text/plain":"<IPython.core.display.HTML object>","text/html":"<blockquote>2. We also obtained the data of cumulative confirmed cases and cumulative tested cases for South Korea from the Korea Centers for Disease Control and Prevention 15, 16 , as shown in Figure 1,.\n</blockquote>"},"metadata":{}}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"model_names = ['en_ner_craft_md','en_ner_jnlpba_md','en_ner_bc5cdr_md','en_ner_bionlp13cg_md']\nmodels = initilize_nlp_models(model_names)\n\nfor task,query in zip(tasks, queries):\n    q_entity_list = []\n    num_docs_found, search_results = search(query, 100)\n    for search_result in search_results:\n        body = search_result.get('body', \"\")\n        for model in models.values(): \n            body_doc = model(body)        \n            q_entity_list = q_entity_list + [e.text for e in body_doc.ents]\n\n    nerCntr = Counter(q_entity_list)\n    freq_ners = nerCntr.most_common(50)\n    print(freq_ners)\n    print(len(freq_ners))\n\n    x,y = zip(*freq_ners)\n    x,y = list(x),list(y)\n\n    plt.figure(figsize=(15,10))\n    ax= sns.barplot(x=x, y=y,palette = sns.cubehelix_palette(len(x)))\n    plt.xlabel('Entity')\n    plt.xticks(rotation=90)\n    plt.ylabel('Frequency')\n    plt.title(task)\n    plt.show()","execution_count":45,"outputs":[{"output_type":"stream","text":"---->  5\n","name":"stdout"}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"def search_custom_query(query):\n    numDocsFound, search_results = search(query)\n    #display(search_results)\n    if numDocsFound > 0:\n        task_answers = search_task_answers(search_results)\n        task_answers = populate_labels(task_answers)\n        #display(task_answers)\n        \n        return task_answers\n\nsearchbar = widgets.interactive(search_custom_query, query='Pregnant women')\nsearchbar","execution_count":51,"outputs":[]},{"metadata":{},"cell_type":"markdown","source":"## Visualization of Force Directed Graph codes"},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"temp_query = searchbar.kwargs\nsearched_query = temp_query['query']\na = search_custom_query(searched_query)\ntask_answers = a","execution_count":50,"outputs":[{"output_type":"error","ename":"KeyError","evalue":"'query'","traceback":["\u001b[0;31m---------------------------------------------------------------------------\u001b[0m","\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)","\u001b[0;32m<ipython-input-50-44fc90b5d8f5>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0mtemp_query\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msearchbar\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearched_query\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtemp_query\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'query'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      3\u001b[0m \u001b[0ma\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msearch_custom_query\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0msearched_query\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0mtask_answers\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0ma\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n","\u001b[0;31mKeyError\u001b[0m: 'query'"]}]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"def cosine_score(x):\n        d = []\n        for i in range(len(x)):\n            for j in range(i+1,len(x)):\n                doc1= generic_model(x[i])\n                doc2= generic_model(x[j])\n                d.append({\n\n                    'Title1': x[i],\n                    'Title2': x[j],\n                    'Score': doc1.similarity(doc2)*30\n                }\n            )\n\n        return d\n\nclass network_graph:\n    \n    def __init__(self,search_result):\n        data = pd.DataFrame(search_result)\n        self.data = data\n        col_list = data.columns\n        drop_list = ['introduction','discussion','result']\n        self.new_cols = []\n        for cols in col_list:\n            if cols not in drop_list:\n                self.new_cols.append(cols)\n\n        \n    def extract_titles(self,search_result):\n        title_list = []\n        for title in range(len(search_result)):\n            title_list.append(search_result[title]['title'])\n            \n        title_dict_temp = cosine_score(title_list)\n        self.title_df = pd.DataFrame(title_dict_temp)\n        title_df_temp1 = self.title_df[['Title1']]\n        title_df_temp1.rename(columns={'Title1':'Title'},inplace=True)\n        title_df_temp2 = self.title_df[['Title2']]\n        title_df_temp2.rename(columns={'Title2':'Title'},inplace=True)\n        self.merged_titles = pd.concat([title_df_temp1,title_df_temp2],axis=0)\n        \n    \n    def extract_words(self,search_result):\n        \n        word_df = self.data[self.new_cols]\n        \n        word_df_temp = word_df.drop(['id','title'],axis=1)\n        \n        col_list = list(word_df_temp.columns)\n        \n        self.word_df_prior =pd.DataFrame()\n        self.word_df_prior_strong =pd.DataFrame()\n        self.word_df_prior_newdata =pd.DataFrame()\n        self.word_df_speculative =pd.DataFrame()\n        self.word_df_unknown =pd.DataFrame()\n        self.word_df_novel =pd.DataFrame()\n        \n        word_df_prior_1 =pd.DataFrame()\n        word_df_prior_strong_1 =pd.DataFrame()\n        word_df_prior_newdata_1 =pd.DataFrame()\n        word_df_speculative_1 =pd.DataFrame()\n        word_df_unknown_1 =pd.DataFrame()\n        word_df_novel_1 =pd.DataFrame()\n            \n        for col in col_list:\n            if col =='prior':\n                self.word_df_prior = word_df[['id','title','prior']]\n                self.word_df_prior = self.word_df_prior.explode('prior')\n                self.word_df_prior.dropna(subset = [\"prior\"], inplace=True)\n                self.word_df_prior['Weight'] = 8\n                \n                word_df_prior_1 = self.word_df_prior[['prior']]\n                word_df_prior_1.drop_duplicates(inplace=True)\n                word_df_prior_1['Color'] = 'tomato'\n                word_df_prior_1['Size'] = 10\n                word_df_prior_1.columns = ['Words','Color','Size']\n                \n            elif col== 'prior-strong':\n                self.word_df_prior_strong = word_df[['id','title','prior-strong']]\n                self.word_df_prior_strong = self.word_df_prior_strong.explode('prior-strong')\n                self.word_df_prior_strong.dropna(subset = [\"prior-strong\"], inplace=True)\n                self.word_df_prior_strong['Weight'] = 6\n                \n                word_df_prior_strong_1 = self.word_df_prior_strong[['prior-strong']]\n                word_df_prior_strong_1.drop_duplicates(inplace=True)\n                word_df_prior_strong_1['Color'] = 'sienna'\n                word_df_prior_strong_1['Size'] = 10\n                word_df_prior_strong_1.columns = ['Words','Color','Size']\n                \n                \n            elif col == 'prior_newdata':\n                self.word_df_prior_newdata = word_df[['id','title','prior_newdata']]\n                self.word_df_prior_newdata = self.word_df_prior_newdata.explode('prior_newdata')\n                self.word_df_prior_newdata.dropna(subset = [\"prior_newdata\"], inplace=True)\n                self.word_df_prior_newdata['Weight'] = 4\n                \n                word_df_prior_newdata_1 = self.word_df_prior_newdata[['prior_newdata']]\n                word_df_prior_newdata_1.drop_duplicates(inplace=True)\n                word_df_prior_newdata_1['Color'] = 'bisque'\n                word_df_prior_newdata_1['Size'] = 10\n                word_df_prior_newdata_1.columns = ['Words','Color','Size']\n                \n                \n            elif col =='speculative':\n                self.word_df_speculative = word_df[['id','title','speculative']]\n                self.word_df_speculative = self.word_df_speculative.explode('speculative')\n                word_df_speculative.dropna(subset = [\"speculative\"], inplace=True)\n                self.word_df_speculative['Weight'] = 10\n                \n                word_df_speculative_1 = self.word_df_speculative[['speculative']]\n                word_df_speculative_1.drop_duplicates(inplace=True)\n                word_df_speculative_1['Color'] = 'dimgray'\n                word_df_speculative_1['Size'] = 10\n                word_df_speculative_1.columns = ['Words','Color','Size']\n                \n            elif col =='unknown':\n                self.word_df_unknown = word_df[['id','title','unknown']]\n                self.word_df_unknown = self.word_df_unknown.explode('unknown')\n                self.word_df_unknown.dropna(subset = [\"unknown\"], inplace=True)\n                self.word_df_unknown['Weight'] = 30\n                \n                word_df_unknown_1 = self.word_df_unknown[['unknown']]\n                word_df_unknown_1.drop_duplicates(inplace=True)\n                word_df_unknown_1['Color'] = 'black'\n                word_df_unknown_1['Size'] = 10\n                word_df_unknown_1.columns = ['Words','Color','Size']\n                \n            elif col =='novel':\n                self.word_df_novel = word_df[['id','title','novel']]\n                self.word_df_novel = self.word_df_novel.explode('novel')\n                self.word_df_novel.dropna(subset = [\"novel\"], inplace=True)\n                self.word_df_novel['Weight'] = 2\n                \n                word_df_novel_1 = self.word_df_novel[['novel']]\n                word_df_novel_1.drop_duplicates(inplace=True)\n                word_df_novel_1['Color'] = 'purple'\n                word_df_novel_1['Size'] = 10\n                word_df_novel_1.columns = ['Words','Color','Size']\n                \n    # Take all the titles and categorize them into one group\n    \n        title_df_1 = self.merged_titles[['Title']]\n        title_df_1.drop_duplicates(inplace=True,keep='first')\n        title_df_1['Color'] = 'firebrick'\n        title_df_1['Size'] = 30\n        title_df_1.columns = ['Words','Color','Size']\n        \n        # Combining all the dataframe together in one place\n\n        frames = [title_df_1,word_df_prior_1,word_df_prior_strong_1,word_df_prior_newdata_1,word_df_speculative_1,word_df_unknown_1,word_df_novel_1]\n         \n        self.merged_df = pd.concat(frames)\n        \n        \n    # Changing the index of merged_df to 'Words' so that can combine it with node\n\n        self.merged_df_index = self.merged_df.drop_duplicates(subset='Words',keep='first')\n        self.merged_df_index.set_index('Words',inplace=True)\n        \n        \n    def network_creation(self):\n        \n        i = nx.Graph()\n        \n        \n\n        if (self.title_df.empty ==False):\n            for row in self.title_df.iterrows():\n                i.add_edge(row[1]['Title1'], row[1]['Title1'], weight=row[1]['Score'])\n\n        if (self.word_df_prior.empty ==False):\n            for row in self.word_df_prior.iterrows():\n                i.add_edge(row[1]['title'], row[1]['prior'], weight=row[1]['Weight'])\n\n        if (self.word_df_prior_strong.empty ==False):\n            for row in self.word_df_prior_strong.iterrows():\n                i.add_edge(row[1]['title'], row[1]['prior-strong'], weight=row[1]['Weight'])\n            \n        if (self.word_df_novel.empty ==False):\n            for row in self.word_df_novel.iterrows():\n                i.add_edge(row[1]['title'], row[1]['novel'], weight=row[1]['Weight'])\n                \n        if (self.word_df_speculative.empty ==False):\n            for row in self.word_df_speculative.iterrows():\n                i.add_edge(row[1]['title'], row[1]['speculative'], weight=row[1]['Weight'])\n                \n        if (self.word_df_prior_newdata.empty ==False):\n            for row in self.word_df_prior_newdata.iterrows():\n                i.add_edge(row[1]['title'], row[1]['prior_newdata'], weight=row[1]['Weight'])\n                \n        if (self.word_df_unknown.empty ==False):\n            for row in self.word_df_unknown.iterrows():\n                i.add_edge(row[1]['title'], row[1]['unknown'], weight=row[1]['Weight'])\n            \n        merged_df_clr = self.merged_df_index.reindex(i.nodes())\n\n        merged_df_clr['Color']=pd.Categorical(merged_df_clr['Color'])\n\n        merged_df_clr['Color'].cat.codes\n        \n        \n        #Plotting the force directed graph\n        \n        plt.figure(figsize=(22, 22))\n        degrees = nx.degree(i)\n        pos_node = nx.spring_layout(i,k=0.5)\n        nx.draw_networkx(i,pos=pos_node,node_color=merged_df_clr['Color'].cat.codes, cmap=plt.cm.Set2,node_size=[(degrees[v] + 1) * 100 for v in i.nodes()],alpha = 0.7)\n        \n    def force_directed_graphs(self):\n        \n        net = Network(notebook=True)\n\n        temp1 = self.merged_df\n        temp2 = temp1.reset_index()\n\n        temp2.drop(columns=['index'],inplace=True)\n\n        net.add_nodes(temp2['Words'],title = temp2['Words'],color=temp2['Color'],size=temp2['Size'].to_list())\n\n        if (self.title_df.empty ==False):\n            for row in self.title_df.iterrows():\n                net.add_edge(row[1]['Title1'], row[1]['Title1'], weight=row[1]['Score'])\n\n        if (self.word_df_prior.empty ==False):\n            for row in self.word_df_prior.iterrows():\n                net.add_edge(row[1]['title'], row[1]['prior'], weight=row[1]['Weight'])\n\n        if (self.word_df_prior_strong.empty ==False):\n            for row in self.word_df_prior_strong.iterrows():\n                net.add_edge(row[1]['title'], row[1]['prior-strong'], weight=row[1]['Weight'])\n            \n        if (self.word_df_novel.empty ==False):\n            for row in self.word_df_novel.iterrows():\n                net.add_edge(row[1]['title'], row[1]['novel'], weight=row[1]['Weight'])\n                \n        if (self.word_df_speculative.empty ==False):\n            for row in self.word_df_speculative.iterrows():\n                net.add_edge(row[1]['title'], row[1]['speculative'], weight=row[1]['Weight'])\n                \n        if (self.word_df_prior_newdata.empty ==False):\n            for row in self.word_df_prior_newdata.iterrows():\n                net.add_edge(row[1]['title'], row[1]['prior_newdata'], weight=row[1]['Weight'])\n                \n        if (self.word_df_unknown.empty ==False):\n            for row in self.word_df_unknown.iterrows():\n                net.add_edge(row[1]['title'], row[1]['unknown'], weight=row[1]['Weight'])\n\n\n        # net.enable_physics(True)\n        display(net.show(\"mygraph.html\"))","execution_count":null,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"#Calling all the functions\nng= network_graph(task_answers)\nng.extract_titles(task_answers)\nng.extract_words(task_answers)\nng.force_directed_graphs()\n# ng.network_creation()","execution_count":null,"outputs":[]},{"metadata":{"trusted":true,"scrolled":false},"cell_type":"code","source":"","execution_count":null,"outputs":[]}],"metadata":{"kernelspec":{"display_name":"Python 3","language":"python","name":"python3"},"language_info":{"name":"python","version":"3.6.4","mimetype":"text/x-python","codemirror_mode":{"name":"ipython","version":3},"pygments_lexer":"ipython3","nbconvert_exporter":"python","file_extension":".py"}},"nbformat":4,"nbformat_minor":4}